Add course about the history of Bitcoin between April 2011 and September 2012 ("The Wild West of finance") (#3348)
DecouvreBitcoin/sovereign-university-data: courses/his203/fr.md

\[À REVOIR (ajouter chapitre minage)\] Dans le prochain chapitre, nous continuerons à traiter la question du développement logiciel, en évoquant l'évolution du logiciel principal. Nous nous concentrerons sur le conflit autour de Pay to Script Hash, une fonctionnalité qui a été ajoutée au protocole en avril 2012.

## L'essor des coopératives de minage

> Le Far West se caractérise aussi par le ruée vers l'or : la découverte d'or en Californie en 1848 avait engendré une grande migration d'aventuriers espérant faire fortune. Bitcoin ayant été associé au métal précieux dès l'origine, il est naturel qu'un tel phénomène se soit développé. C'est ainsi que l'activité du minage de bitcoins a attiré des individus passionnés qui ont, par leur intérêt pécuniaire, contribué à la rendre plus efficace au cours du temps.

Reviewer: ```suggestion
Le Far West se caractérise aussi par la ruée vers l'or : la découverte d'or en Californie en 1848 avait engendré une grande migration d'aventuriers espérant faire fortune. Bitcoin ayant été associé au métal précieux dès son origine, il est naturel qu'un tel phénomène se soit développé. C'est ainsi que l'activité du minage de bitcoins a attiré des individus passionnés qui ont, par leur intérêt pécuniaire, contribué à la rendre plus efficace au cours du temps.
```

> À l'automne 2010, une première « ruée vers l'or numérique » (original: "digital gold rush") avait eu lieu, et avait entrainé une généralisation du minage par processeur graphique (GPU), beaucoup plus rentable que le minage par processeur central (CPU). Cette amélioration avait cependant fait augmenter la difficulté à un niveau tel que les petits mineurs ne pouvaient plus trouver de bloc dans un temps acceptable. C'est pourquoi les premiers regroupements de mineurs sont apparus, dont notamment BitcoinCZ Mining, la coopérative de Slush.

Reviewer: ```suggestion
À l'automne 2010, une première « ruée vers l'or numérique » (original: "digital gold rush") avait eu lieu, et avait entrainé une généralisation du minage par processeur graphique (GPU), beaucoup plus rentable que le minage par processeur central (CPU). Cette amélioration avait cependant fait augmenter la difficulté à un niveau tel que les petits mineurs ne pouvaient plus espérer trouver un bloc dans un délai raisonnable. C'est pourquoi les premiers regroupements de mineurs sont apparus, dont notamment BitcoinCZ Mining, la coopérative de Slush.
```

### L'augmentation du taux de hachage

Avec la généralisation du minage par processeur graphique et la croissance du taux de change record se produisant entre 2010 et 2011, le taux de hachage du réseau monte de façon exponentielle. Il est ainsi multiplié par 1 000 en moins d'un an. Alors qu'il n'était que de 10 gigahachages par seconde (GH/s) en septembre 2010, il atteint 100 GH/s en décembre, puis 1 TH/s en mai 2011 et dépasse les 10 TH/s en juin. Il culmine à 15,8 TH/s avant de redescendre de façon abrupte, conséquemment à l'éclatement de la grande bulle de 2011. Après avoir touché un point bas à 7 GH/s en décembre, il se stabilise entre 10 et 15 GH/s lors du premier semestre de 2012.

![Taux de hachage total du réseau entre avril 2011 et juillet 2012](assets/en/btc-hashrate-coinwarz-20110420-20110720.webp)

Taux de hachage total du réseau de Bitcoin entre avril 2011 et juillet 2012 (source : [CoinWarz](https://www.coinwarz.com/mining/bitcoin/hashrate-chart))

> Cette augmentation de la puissance de calcul du réseaux fait que le « minage en solitaire » (*solo mining*) n'est plus viable du tout, la variance étant trop grande la majorité des installations. C'est pourquoi la plupart des mineurs sont obligés de se réunir en coopératives de minage, ou *mining pools* en anglais, afin de lisser leurs revenus. Ils louent leur puissance de hachage à un opérateur, qui s'occupe de choisir les spécificités des blocs et les transactions à inclure.

Reviewer: ```suggestion
Cette augmentation de la puissance de calcul du réseau fait que le « minage en solitaire » (*solo mining*) n'est plus viable du tout, la variance étant trop grande pour la majorité des installations. C'est pourquoi la plupart des mineurs sont obligés de se réunir en coopératives de minage, ou *mining pools* en anglais, afin de lisser leurs revenus. Ils louent leur puissance de hachage à un opérateur, qui s'occupe de choisir les spécificités des blocs et les transactions à inclure.
```

C'est dans ce contexte qu'en décembre 2010, Marek Palatinus (alias Slush) [lance](https://bitcointalk.org/index.php?topic=1976.msg30520#msg30520) BitcoinCZ Mining, un service de « minage coopératif » (original: "cooperative mining") qui recueille 3,5 % du taux de hachage du réseau en quelques jours. Ce succès ne manque pas de faire des émules, et l'année 2011 est marquée par une normalisation du minage groupé, qui s'illustre par l'ouverture d'un [grand nombre](https://en.bitcoin.it/w/index.php?title=Comparison_of_mining_pools&oldid=21595) de coopératives.

### Les coopératives de minage et leur mode de rémunération

> Différentes [méthodes de rémunération](https://arxiv.org/abs/1112.4980) sont adoptées par les coopératives de minages. Le calcul est basé sur les « parts » (original: "shares") fournies par les clients, c'est-à-dire les preuves de travail partielles calculées pour un bloc donné. Initialement, la coopérative de Slush applique un modèle proportionnel simple : ses utilisateurs sont récompensés à chaque bloc miné, au pro rata des parts qu'ils ont trouvées depuis le dernier bloc. Elle prélève sur cette récompense une commission de 2 %, soit 1 BTC par bloc. Ce modèle est cependant légèrement défectueux, les mineurs [pouvant](https://bitcointalk.org/index.php?topic=3165.msg44357#msg44357) « tricher » (et obtenir environ 28 % de bitcoins en plus) en se déconnectant de la coopérative une fois un certain nombre de parts soumises lors d'un cycle et en redirigeant leur puissance de calcul à leur propre compte ou vers une autre coopérative (une pratique [connue](https://bitcointalk.org/index.php?topic=26866.msg337932#msg337932) sous le nom de *pool hopping*). C'est pourquoi Slush [met en place](https://bitcointalk.org/index.php?topic=1976.msg50002#msg50002), le 4 février 2011, un système de score « anti-tricherie » (original: "cheat proof") faisant que les parts les plus récentes du cycle ont plus de poids que les parts les plus anciennes.

Reviewer: ```suggestion
Différentes [méthodes de rémunération](https://arxiv.org/abs/1112.4980) sont adoptées par les coopératives de minage. Le calcul est basé sur les « parts » (original: "shares") fournies par les clients, c'est-à-dire les preuves de travail partielles calculées pour un bloc donné. Initialement, la coopérative de Slush applique un modèle proportionnel simple : ses utilisateurs sont récompensés à chaque bloc miné, au pro rata des parts qu'ils ont trouvées depuis le dernier bloc. Elle prélève sur cette récompense une commission de 2 %, soit 1 BTC par bloc. Ce modèle est cependant légèrement défectueux, les mineurs [pouvant](https://bitcointalk.org/index.php?topic=3165.msg44357#msg44357) « tricher » (et obtenir environ 28 % de bitcoins en plus) en se déconnectant de la coopérative une fois un certain nombre de parts soumises lors d'un cycle et en redirigeant leur puissance de calcul à leur propre compte ou vers une autre coopérative (une pratique [connue](https://bitcointalk.org/index.php?topic=26866.msg337932#msg337932) sous le nom de *pool hopping*). C'est pourquoi Slush [met en place](https://bitcointalk.org/index.php?topic=1976.msg50002#msg50002), le 4 février 2011, un système de score « anti-tricherie » (original: "cheat proof") faisant que les parts les plus récentes du cycle ont plus de poids que les parts les plus anciennes.
```

Reviewer: J'ai supprimé le "s" à "minage" dans la première phrase (github commence à halluciner, il ne marque plus mes changements ^^)

Le 27 février, une nouvelle coopérative est [ouverte](https://web.archive.org/web/20111127072845/https://bitcointalk.org/index.php?topic=3889.0) par un [Russe](https://buildingbitcoin.org/bitcoin-dev/log-2011-02-19.html#l-2646) se faisant appeler Tycho (qu'il orthographie \[Tycho\]). Il s'agit de Deepbit, qui a récompense ses clients à chaque bloc miné, au pro rata des parts qu'ils ont trouvées depuis le dernier bloc. À ce modèle proportionnel, Tycho [ajoute](https://bitcointalk.org/index.php?topic=3889.msg55799#msg55799) rapidement le modèle PPS à son offre, mais prélève des frais supérieurs pour ce mode (10 % au lieu de 3 %).

> Par la suite, plusieurs de dizaines coopératives (plus ou moins sérieuses) sont ouvertes dans l'année. Parmi elles, on trouve BTC Guild, une coopérative installée aux États-Unis qui est [créée](https://web.archive.org/web/20111127113730/https://bitcointalk.org/index.php?topic=7760.0) en mai 2011 par un mineur californien se faisant appeler Eleuthria (déformation du mot grec ἐλευθερία, eleuthería, qui signifie « liberté »). Elle applique un modèle PPS pur.

Reviewer: ```suggestion
Par la suite, plusieurs dizaines de coopératives (plus ou moins sérieuses) sont ouvertes dans l'année. Parmi elles, on trouve BTC Guild, une coopérative installée aux États-Unis qui est [créée](https://web.archive.org/web/20111127113730/https://bitcointalk.org/index.php?topic=7760.0) en mai 2011 par un mineur californien se faisant appeler Eleuthria (déformation du mot grec `ἐλευθερία`, `eleuthería`, qui signifie « liberté »). Elle applique un modèle PPS pur.
```

Ici, j'ai ajouté du code inline pour être sûr que ce mot (qui doit toujours rester comme dans l'original) ne soit pas traduit dans les autres langues par la suite. Le code inline n'est jamais traduit. Si ça te pose un problème niveau mise en forme, on peut l'enlever et on demandera aux relecteurs d'être vigilant ici. Comme tu préfères

Des coopératives pour Namecoin apparaissent aussi, comme [Bitparking](https://bitcointalk.org/index.php?topic=8775.msg127063#msg127063), ouverte par DoubleC en mai, ou [NMCBit](https://bitcointalk.org/index.php?topic=33612.msg419958#msg419958), créée par un certain Davinci Jeremie (DavinciJ15) en août. L'ajout du minage combiné fera qu'au fur et à mesure du temps, toutes les coopératives de Bitcoin seront amenées à intégrer le minage de Namecoin à leur offre.

> Des méthodes de versement plus complexes ayant pour but de corriger les déficiences des modèles simples sont conceptualisés : le PPLNS, la méthode géométrique, et le SMPPS.

Reviewer: ```suggestion
Des méthodes de versement plus complexes ayant pour but de corriger les déficiences des modèles simples sont conceptualisées : le PPLNS, la méthode géométrique, et le SMPPS.
```

D'autres coopératives sauront tirer leur épingle du jeu. On peut citer le groupe minier américain BitMinter, [lancé](https://bitcointalk.org/index.php?topic=22432.msg281686#msg281686) en juin, qui a la particularité de proposer aussi un logiciel de minage. On peut mentionner également la coopérative russe 50BTC.com, [créée](https://bitcointalk.org/index.php?topic=54673.msg651025#msg651025) en décembre, qui fournira un outil de surveillance d'activité pour les mineurs appelé Mining Explorer.

![L'outil Mining Explorer en mai 2012](assets/en/mining-explorer-50btc-may-2012.webp)

*L'outil Mining Explorer en mai 2012 (source : [archive](https://web.archive.org/web/20120609002559/http://miningexplorer.org/) de Miningexplorer.org)*

### Eligius, la coopérative de Luke-Jr

> Une coopérative de minage originale est Eligius, qui est ouverte par le dévelopeur Luke-Jr en avril 2011. Luke-Jr (ou Luke Dashjr, prononcé « dasheur ») est un personnage atypique de l'histoire de Bitcoin. Fervent catholique romain [sédévacantiste](https://dashjr.org/catholic/sedevacantist.php), il est marié et père d'une famille nombreuse (il a deux enfants en 2007 et en aura trois de plus en 2017). Il est programmeur dans le domaine du logiciel libre, ayant développé son propre système d'exploitation, appelé [Utopios](https://web.archive.org/web/20041013023456/http://utopios.org/) et basé sur la distribution Gentoo.

Reviewer: ```suggestion
Une coopérative de minage originale est Eligius, qui est ouverte par le développeur Luke-Jr en avril 2011. Luke-Jr (ou Luke Dashjr, prononcé « dasheur ») est un personnage atypique de l'histoire de Bitcoin. Fervent catholique romain [sédévacantiste](https://dashjr.org/catholic/sedevacantist.php), il est marié et père d'une famille nombreuse (il a deux enfants en 2007 et en aura trois de plus en 2017). Il est programmeur dans le domaine du logiciel libre, ayant développé son propre système d'exploitation, appelé [Utopios](https://web.archive.org/web/20041013023456/http://utopios.org/) et basé sur la distribution Gentoo.
```

Eligius innove aussi par la pratique de la « signature de la coinbase » (original: "coinbase signing"), qui consiste à écrire le nom de la coopérative (`Eligius` dans ce cas) dans le champ d'entrée de la transaction de récompense du bloc miné. Cette pratique, permettant de rendre plus transparent le fonctionnement des coopératives, est inaugurée le jour du passage vers le modèle MPPS, dans la nuit du 13 au 14 juin, avec le bloc [130 635](https://mempool.space/block/0000000000000a067d94ff753eec72830f1205ad3a4c216a08a80c832e551a52), qui contient . Il s'agit de la deuxième inscription réalisée par ce moyen, après l'emblématique citation du *Times* incluse par Satoshi Nakamoto dans le bloc de genèse.

> À partir du 5 août, Luke-Jr utilise ce champ pour [ajouter](https://ourbigbook.com/cirosantilli/cool-data-embedded-in-the-bitcoin-blockchain/prayer-wars) des messages de nature religieuse des blocs générés par la coopérative. Conformément à la foi catholique traditionnaliste de l'opérateur, on y retrouve des prières en latin : les Louanges divines, le *Je vous salue Marie*,le *Salve Regina*, le *Notre Père*, la prière du cœur, etc. Mais il y a aussi des textes en anglais : les Actes de contrition, de foi, d'espérance et de charité, le Confiteor, la prière à l'Ange Gardien, des méditations du chapelet du Sacré-Cœur de Jésus, des passages de l'Évangile, des commémorations des saints (comme celle d'Augustin d'Hippone le 28 août).

Reviewer: ```suggestion
À partir du 5 août, Luke-Jr utilise ce champ pour [ajouter](https://ourbigbook.com/cirosantilli/cool-data-embedded-in-the-bitcoin-blockchain/prayer-wars) des messages de nature religieuse dans les blocs générés par la coopérative. Conformément à la foi catholique traditionnaliste de l'opérateur, on y retrouve des prières en latin : les Louanges divines, le *Je vous salue Marie*, le *Salve Regina*, le *Notre Père*, la prière du cœur, etc. Mais il y a aussi des textes en anglais : les Actes de contrition, de foi, d'espérance et de charité, le Confiteor, la prière à l'Ange Gardien, des méditations du chapelet du Sacré-Cœur de Jésus, des passages de l'Évangile, des commémorations des saints (comme celle d'Augustin d'Hippone le 28 août).
```

### La centralisation de l'activité minière

Le regroupement en coopératives pose un problème de centralisation, car la construction des blocs est déléguée à un opérateur, qui se charge de choisir les paramètres et de sélectionner les transactions à confirmer. De plus, les plus grosses coopératives sont ainsi favorisées au détriment des plus petites, car la variance s'applique également aux groupes de mineurs, qui ont besoin d'être suffisamment importants pour ne pas en subir les effets. La centralisation du minage qui résulte de cette situation inquiète naturellement les membres de la communauté, qui la considèrent comme une menace vis-à-vis du bon fonctionnement de Bitcoin.

> L'activité transparente des coopératives publiques (comme par exemple Deepbit qui [publie](https://web.archive.org/web/20110602135526/http://deepbit.net/stats) ses blocs trouvés tous les jours) rend possible d'estimer la proportion de leur taux de hachage par rapport à la puissance totale du réseau. C'est ce qui permet de générer des graphique en secteurs illustrant la distribution des coopératives, comme le font par exemple [Bitcoin Watch](https://web.archive.org/web/20110401115146/http://www.bitcoinwatch.com/) (alors [géré](https://bitcointalk.org/index.php?topic=5016.msg73505#msg73505) par Nils Schneider) ou [Blockchain.info](https://web.archive.org/web/20111018232813/http://blockchain.info/pools). Il est ainsi possible de s'assurer que le taux de hachage ne soit pas centralisé au sein d'une seule coopérative.

Reviewer: ```suggestion
L'activité transparente des coopératives publiques (comme par exemple Deepbit qui [publie](https://web.archive.org/web/20110602135526/http://deepbit.net/stats) ses blocs trouvés tous les jours) rend possible d'estimer la proportion de leur taux de hachage par rapport à la puissance totale du réseau. C'est ce qui permet de générer des graphiques en secteurs illustrant la distribution des coopératives, comme le font par exemple [Bitcoin Watch](https://web.archive.org/web/20110401115146/http://www.bitcoinwatch.com/) (alors [géré](https://bitcointalk.org/index.php?topic=5016.msg73505#msg73505) par Nils Schneider) ou [Blockchain.info](https://web.archive.org/web/20111018232813/http://blockchain.info/pools). Il est ainsi possible de s'assurer que le taux de hachage ne soit pas centralisé au sein d'une seule coopérative.
```

![Distribution du taux de hachage de Bitcoin le 6 juillet 2011](assets/en/hashrate-distribution-pie-20110706bitcoinwatch.webp)

*Distribution du taux de hachage de Bitcoin le 6 juillet 2011 (source : [archive](https://web.archive.org/web/20110706215910/http://www.bitcoinwatch.com/) de Bitcoin Watch)*

> Et c'est ce qui se produit au début du mois de juillet, lorsque la part du taux de hachage mesuré de Deepbit, la coopérative de Tycho, avoisine les 50 % (à court terme). Cette domination provoque la réaction de certaines personnes sur le forum [pour qui](https://bitcointalk.org/index.php?topic=26630.msg333929#msg333929) « la sécurité de Bitcoin \[est\] presque compromise » (original: "Bitcoin Security nearly Breached"). Les utilisateurs les plus informés sont agacés, et rappellent [que](https://bitcointalk.org/index.php?topic=26630.msg333929#msg333929) l'opérateur n'a aucun intérêt économique à attaquer le réseau, que l'impact de cette attaque serait limité et [que](https://bitcointalk.org/index.php?topic=26630.msg334029#msg334029) les hacheurs sont à tout moment libres de changer de service, puisque « l'opérateur de la coopérative ne contrôle pas le matériel informatique de ses membres » (original: "The pool operator does not control the hardware of the members of the pool"). Il y a donc un problème de centralisation mais celui-ci n'est pas aussi grave qu'il n'en a l'air.

Reviewer: ```suggestion
Et c'est ce qui se produit au début du mois de juillet, lorsque la part du taux de hachage mesuré de Deepbit, la coopérative de Tycho, avoisine les 50 % (à court terme). Cette domination provoque la réaction de certaines personnes sur le forum [pour qui](https://bitcointalk.org/index.php?topic=26630.msg333929#msg333929) « la sécurité de Bitcoin \[est\] presque compromise » (original: "Bitcoin Security nearly Breached"). Les utilisateurs les plus informés sont agacés, et rappellent [que](https://bitcointalk.org/index.php?topic=26630.msg333929#msg333929) l'opérateur n'a aucun intérêt économique à attaquer le réseau, que l'impact de cette attaque serait limité et [que](https://bitcointalk.org/index.php?topic=26630.msg334029#msg334029) les hacheurs sont à tout moment libres de changer de service, puisque « l'opérateur de la coopérative ne contrôle pas le matériel informatique de ses membres » (original: "The pool operator does not control the hardware of the members of the pool"). Il y a donc un problème de centralisation, mais celui-ci n'est pas aussi grave qu'il n'y paraît.
```

Reviewer: modification de la dernière phrase : ajout de virgule + "qu'il n'y paraît"

![Forrest Voight vers 2011](assets/en/forrest-voight-circa-2011-forre-st.webp)

*Forrest Voight vers 2011 (source : [archive](https://web.archive.org/web/20110816144339/https://forre.st/) de son site personnel)*

Le modèle [repose](https://im.forre.st/pb/85341005.txt) sur une « chaine de parts », ou *sharechain*, qui est maintenue par les mineurs participants connectés en pair à pair, à l'instar de la chaine de blocs de Bitcoin. Chaque part contient l'empreinte de la part précédente, ce qui lie les parts entre elles et assure l'intégrité de l'ensemble. La difficulté (qui est initialement 600 fois inférieure à la difficulté du réseau principal) est ajustée régulièrement pour garantir un intervalle moyen de 5 secondes. Le mode de rémunération est le paiement selon les N dernières parts (PPLNS), réalisé directement dans la transaction de récompense, à chaque fois qu'un bloc est trouvé.

![Visualisation de la share chain](assets/en/p2pool-share-chain-gregory-maxwell-20120108.webp)

*Visualisation de la sharechain (source : Gregory Maxwell pour [Bitcoin Wiki](https://en.bitcoin.it/wiki/File:P2pool_chain.png))*

> Le 19 juillet 2011, après un essai réalisé sur le réseau de test, P2Pool est [déployée](https://bitcointalk.org/index.php?topic=61131.msg379115#msg379115) sur le réseau principal de Bitcoin. La coopérative décentralisée [trouve](https://bitcointalk.org/index.php?topic=61131.msg476766#msg476766) son premier bloc le 23 août, après plusieurs mises à niveau et plus d'un mois de hachage ! D'autres blocs suivront mais l'activité restera anecdotique durant des mois.

Reviewer: ```suggestion
Le 19 juillet 2011, après un essai réalisé sur le réseau de test, P2Pool est [déployée](https://bitcointalk.org/index.php?topic=61131.msg379115#msg379115) sur le réseau principal de Bitcoin. La coopérative décentralisée [trouve](https://bitcointalk.org/index.php?topic=61131.msg476766#msg476766) son premier bloc le 23 août, après plusieurs mises à niveau et plus d'un mois de hachage ! D'autres blocs suivront, mais l'activité restera anecdotique durant des mois.
```

À partir du mois de décembre, le système s'améliore et attire l'attention de la communauté technique, notamment [Mike Hearn](https://bitcointalk.org/index.php?topic=61131.msg641090#msg641090), [Gregory Maxwell](https://bitcointalk.org/index.php?topic=18313.msg712952#msg712952) et [Charlie Lee](https://bitcointalk.org/index.php?topic=18313.msg712976#msg712976). Le protocole permet de réaliser des dons pour encourager les mineurs à utiliser la coopérative, ce que certains [font](https://bitcointalk.org/index.php?topic=57027.msg679955#msg679955). En raison de cette attention, le taux de hachage géré par la coopérative [décolle](https://organofcorti.blogspot.com/2012/11/52-p2pool-achieving-expectations.html) enfin : il passe de quelques GH/s début 2012, à 100 GH/s en février et 350 GH/s à la fin du mois de mars. Le système représente alors 3 % de la puissance de calcul totale du réseau ! Toutefois, P2Pool peinera à se faire une place plus grande, son fonctionnement étant par essence plus complexe que les autres coopératives.

### Le protocole de minage Stratum

> La généralisation du minage en coopératives motive également le développement de nouveaux protocoles de communication. En 2011, les coopératives reposaient sur la rudimentaire commande `getwork`, ainsi que sur le *long polling*, une [technique](https://bitcointalk.org/index.php?topic=3889.msg63957#msg63957) issue de la programmation réseau permettant (entre autres) de réduire les parts obsolètes en annonçant rapidement les nouveaux blocs aux mineurs. Cependant, ce fonctionnement est extrêmement coûteux en bande passante, que ce soit du côté du serveur que du client, si bien que la situation devient intenable à mesure que le nombre de connexions explose.

Reviewer: ```suggestion
La généralisation du minage en coopératives motive également le développement de nouveaux protocoles de communication. En 2011, les coopératives reposaient sur la rudimentaire commande `getwork`, ainsi que sur le *long polling*, une [technique](https://bitcointalk.org/index.php?topic=3889.msg63957#msg63957) issue de la programmation réseau permettant (entre autres) de réduire les parts obsolètes en annonçant rapidement les nouveaux blocs aux mineurs. Cependant, ce fonctionnement est extrêmement coûteux en bande passante, que ce soit du côté du serveur ou du client, si bien que la situation devient intenable à mesure que le nombre de connexions explose.
```

![Bande dessinée de XKCD partagée par Luke-Jr en réaction au développement du nouveau standard Stratum](assets/en/xkcd-standards-20110720.webp)

« Comment les normes se multiplient » : bande dessinée de XKCD partagée par Luke-Jr en réaction au développement du nouveau standard Stratum (source : [XKCD](https://xkcd.com/927/)) -- FR/EN: https://xkcd.arnaud.at/927

Au fil des mois, Stratum s'impose comme le protocole utilisé par la plupart des coopératives, à commencer par BitcoinCZ Mining et BTC Guild. Le taux de hachage utilisant Stratum représente [24 %](https://web.archive.org/web/20121022072300/https://mining.bitcoin.cz/stats/) en octobre, [45 %](https://web.archive.org/web/20121125081858/https://mining.bitcoin.cz/stats/) en novembre, et atteint [74 %](https://web.archive.org/web/20121225110316/https://mining.bitcoin.cz/stats/) en décembre ! La fonction `getwork` sera progressivement [abandonnée](https://bitcointalk.org/index.php?topic=1976.msg1524107#msg1524107) au cours de l'année 2013.

### Le minage par FPGA et par ASIC

> L'organisation du minage a donc considérablement évolué au cours des années 2011–2012, en étant gérée par des coopératives plutôt que par des mineurs en solitaire. On a assisté à un véritable essor des coopératives, celles-ci étant très diverses tant par leur philosophie que par leur méthode de rémunération (proportionnelle, PPS, etc.) Ce changement a cependant provoqué une centralisation de l'activité minière, qui motivé la création du modèle de coopérative décentralisée P2Pool en juillet 2011. La communication au sein des coopératives a aussi connu une amélioration, qui s'est conclue par l'apparition du protocole léger Stratum en septembre 2012.

Reviewer: ```suggestion
L'organisation du minage a donc considérablement évolué au cours des années 2011–2012, en étant gérée par des coopératives plutôt que par des mineurs en solitaire. On a assisté à un véritable essor des coopératives, celles-ci étant très diverses tant par leur philosophie que par leur méthode de rémunération (proportionnelle, PPS, etc.) Ce changement a cependant provoqué une centralisation de l'activité minière, qui a motivé la création du modèle de coopérative décentralisée P2Pool en juillet 2011. La communication au sein des coopératives a aussi connu une amélioration, qui s'est conclue par l'apparition du protocole léger Stratum en septembre 2012.
```

> Évidemment, l'évolution du minage n'a pas concerné que l'organisation des hacheurs ; elle a aussi été celle du matériel lui-même. Si le passage du processeur central (CPU) au processeur graphique (GPU) avait déjà été une fracture, il ne faisait que préfigurer l'apparition des circuits intégrés spécialisés, ou ASIC, qui ferait faire un nouveau bond de géant à la puissance de calcul du réseau.

Reviewer: ```suggestion
Évidemment, l'évolution du minage n'a pas concerné que l'organisation des hacheurs ; elle a aussi été celle du matériel lui-même. Si le passage du processeur central (CPU) au processeur graphique (GPU) avait déjà été une rupture, il ne faisait que préfigurer l'apparition des circuits intégrés spécialisés, ou ASIC, qui ferait faire un nouveau bond de géant à la puissance de calcul du réseau.
```

> Cette évolution finale, [prévue](https://www.metzdowd.com/pipermail/cryptography/2008-November/014815.html) par Satoshi Nakamoto de longue date, a été progressivement développée grâce aux circuits logiques programmables (FPGA), qui permettaient d'optimiser virtuellement le matériel pour la fonction de hachage de Bitcoin. Le premier algorithme ouvert pour FPGA a été [publié](https://bitcointalk.org/index.php?topic=9047.msg130885#msg130885) en mai 2011. Les ASIC ont appliqué les optimisations des FPGA aux puces elles-mêmes. La première prévente d'ASIC, réalisée par la société étasunienne Butterfly Labs, a [eu lieu](https://web.archive.org/web/20120619090952/http://news.yahoo.com/butterfly-labs-announces-next-generation-asic-lineup-054626776.html) en juin 2012. Enfin, le premier ASIC fonctionnel, l'Avalon ASIC, a été [distribué](https://bitcointalk.org/index.php?topic=137534.msg1465887#msg1465887) au grand public en janvier 2013.

Reviewer: ```suggestion
Cette évolution finale, [prévue](https://www.metzdowd.com/pipermail/cryptography/2008-November/014815.html) de longue date par Satoshi Nakamoto, a été progressivement développée grâce aux circuits logiques programmables (FPGA), qui permettaient d'optimiser virtuellement le matériel pour la fonction de hachage de Bitcoin. Le premier algorithme ouvert pour FPGA a été [publié](https://bitcointalk.org/index.php?topic=9047.msg130885#msg130885) en mai 2011. Les ASIC ont appliqué les optimisations des FPGA aux puces elles-mêmes. La première prévente d'ASIC, réalisée par la société étasunienne Butterfly Labs, a [eu lieu](https://web.archive.org/web/20120619090952/http://news.yahoo.com/butterfly-labs-announces-next-generation-asic-lineup-054626776.html) en juin 2012. Enfin, le premier ASIC fonctionnel, l'Avalon ASIC, a été [distribué](https://bitcointalk.org/index.php?topic=137534.msg1465887#msg1465887) au grand public en janvier 2013.
```

# La construction de Bitcoin entre 2011 et 2012 : le Far West de la finance

> Titre alternatif : Far West de la finance -> Far West de la cryptomonnaie ?

Reviewer: Idée : mettre un titre assez simple, court et direct, et utiliser cette expression "Far West de la finance" en sous-titre plutôt. Je trouve cela très cool : ça donne envie de lire.

Author: « La construction de Bitcoin entre 2011 et 2012 » est correct comme titre ?

> « Tout à fait, Bitcoin est le Far West de la finance, et Dieu merci. Il représente toute une cohorte d'aventuriers et d'entrepreneurs, de preneurs de risques, d'inventeurs et de résolveurs de problèmes. C'est la Frontière. D'énormes quantités de richesses seront créées et détruites au fur et à mesure que ce nouveau paysage se dessinera. Je pense que les effets de cette aventure seront considérables, car si le "Far West" a été un phénomène exclusivement américain, Bitcoin est lui un mouvement mondial. »
>

> > original: "Bitcoin is absolutely the Wild West of finance, and thank goodness. It represents a whole legion of adventurers and entrepreneurs, of risk takers, inventors, and problem solvers. It is the frontier. Huge amounts of wealth will be created and destroyed as this new landscape is mapped out. I believe the effects of this adventure will be profound, for while the 'Wild West' was a uniquely American phenomenon, Bitcoin is a global one."

Reviewer: Sur les citations originales, il faudra voir comment on gère cela lors du processus de traduction : 
- quand on va traduire en anglais, ça va faire un doublon
- quand on va traduire de l'anglais vers toutes les autres langues, l'original sera lui aussi traduit.

À voir avec Asi0 comment on gère cela : à la main si y'en a pas beaucoup c'est gérable, ou alors c'est typiquement le genre de truc qu'on peut faire avec Polyglotter en post traduction, mais le fait que l'anglais seul soit différent (puisque pas de notion original/traduction), ça va casser la cohérence des paragraphes entre les langues).

Reviewer: Update: y'en a vraiment beaucoup, et certaines ne sont pas des paragraphes indépendants, mais sont intégrées dans du texte. Le plus simple serait de les laisser comme ça en français, et on demande au premier relecteur en anglais de conserver uniquement les originales. Les autres langues n'auront pas la version originale en anglais, uniquement la version traduite, mais sinon ça va être très chaud de gérer ça manuellement.

Autre solution : ne rien faire, et demander aux relecteurs dans chaque langue d'être attentifs à cela, et de remettre l'original en anglais s'il a été traduit par le script (mais ça demande beaucoup de boulot).

À voir avec Mari et Asi0.

Author: Le traitement de ces citations originales avait été faite automatiquement pour HIS201, il me semble.

Reviewer: Ok alors Asi0 doit avoir une technique. On verra avec lui lors de la publication

L'élément marquant est probablement l'âge des principaux promoteurs de Bitcoin lors de cette période. La plupart d'entre eux étaient jeunes, voire très jeunes ; c'étaient pour une bonne partie des *millenials*, des membres de la génération Y : nés entre 1981 et 1996, ils avaient entre 16 et 31 ans en 2011 lorsque Satoshi a disparu définitivement. Leur jeunesse relative les a rendu particulièrement enthousiastes, ce qui fait qu'ils ont constitué en quelque sorte le carburant avec lequel Bitcoin a pu prendre son envol. Néanmoins, ils étaient aussi particulièrement immatures et imprudents, notamment vis-à-vis de la réponse réglementaire. Même s'ils étaient encadrés par les membres de la génération précédente (souvent plus discrets), ils ont ainsi fait un nombre considérable d'erreurs, qui ont mené certains d'entre eux à subir la répression des autorités ou à finir derrière les barreaux.

### Plan du cours

### Précisions

La lecture du premier cours sur l'histoire de Bitcoin (intitulée *L'histoire de la création de Bitcoin*) est un prérequis si vous ne connaissez pas du tout le sujet.

> https://planb.network/courses/lhistoire-de-la-creation-de-bitcoin-a51c7ceb-e079-4ac3-bf69-6700b985a082

Reviewer: ```suggestion
https://planb.network/courses/a51c7ceb-e079-4ac3-bf69-6700b985a082
```

Juste avec l'UUID ça suffit

- [*Deep Web*](https://www.youtube.com/watch?v=L7emWKAMEvI), réalisé par Alex Winter, 31 mai 2015
- [*Banking on Bitcoin*](https://www.youtube.com/watch?v=8rsxIW02W0g), réalisé par Christopher Cannucciari, 2016

# La grande bulle de 2011

## Silk Road, l'Amazon de la drogue (janv.–nov. 2011)

La question de la relation entre Bitcoin et l'autorité s'est posée assez rapidement dans l'histoire de Bitcoin. En décembre 2010, lorsque Satoshi Nakamoto s'est retiré des devants de la scène, sa raison était la publication d'un article sur PC World qui invitait WikiLeaks à utiliser Bitcoin. Comme pour confirmer son intuition, cet article a attiré une attention monstrueuse sur Bitcoin, dont le prix est passé de 0,20 $ à 32 $ en quelques mois. Satoshi a continué d'échanger en privé avec ses plus proches collaborateurs et a définitivement disparu au cours du printemps 2011. Assez symboliquement, le dernier message de Gavin Andresen, le développeur qui avait pris les rênes du projet, informait le créateur de Bitcoin qu'il avait été contacté par In-Q-Tel, un fonds d'investissement géré par la CIA, ce qui démontrait que la boite de Pandore était ouverte.

> Satoshi Nakamoto a eu raison de s'effacer à ce moment-là, car les années qui ont suivi ont été marquée par la prolifération des utilisations illégales de Bitcoin. Très logiquement, les hors-la-loi avaient à un intérêt à se servir de la résistance à la censure de Bitcoin pour échapper au contrôle de l'État. L'usage le plus populaire était probablement le trafic de drogue en ligne, qui se faisait par le biais d'une place de marché novatrice : Silk Road.

Reviewer: ```suggestion
Satoshi Nakamoto a eu raison de s'effacer à ce moment-là, car les années qui ont suivi ont été marquées par la prolifération des utilisations illégales de Bitcoin. Très logiquement, les hors-la-loi avaient un intérêt à se servir de la résistance à la censure de Bitcoin pour échapper au contrôle de l'État. L'usage le plus populaire était probablement le trafic de drogue en ligne, qui se faisait par le biais d'une place de marché novatrice : Silk Road.
```

Cette plateforme, surnommée assez rapidement l'Amazon de la drogue, a été créée par un jeune texan de 26 ans du nom de Ross Ulbricht. Ce dernier possédait des idéaux libertariens prononcés, qui lui avait inspiré l'idée de son projet. Silk Road a connu un succès fulgurant et a fini par occuper une place de choix dans le paysage de Bitcoin, si bien que nous lui consacrons ici un chapitre entier. Nous parlerons d'abord de Ross Ulbricht, le cerveau derrière Silk Road, puis des débuts de la place de marché, avant d'évoquer ses premiers succès.

### Ross Ulbricht, le libertarianisme et l'agorisme

Ross William Ulbricht grandit à Austin, dans l'État du Texas aux États-Unis, où il passe une jeunesse sans histoires, entouré d'une famille aimante. Il est scout durant de nombreuses années, ce qui lui donne goût pour les activités en extérieur, notamment la randonnée et le camping, et pour le sport. Plutôt brillant, il bénéficie d'une bourse d'études complète pour aller à l'université, où il obtient une licence de physique et un master de science et ingénierie des matériaux spécialisé en cristallographie.

![Ross Ulbricht](assets/en/ross-ulbricht-tree.webp)

> Ross Ulbricht à une date indéterminée (source : FreeRoss.org)

Reviewer: On n'a pas encore de système standard sur la plateforme pour intégrer les légendes. En attendant, est-ce qu'on ne mettrais pas les légendes de ton cours en italique pour qu'elles ressortent par rapport aux paragraphes ?

Author: Mis en italiques. 4cb3fdc93aca57666bf32e464b8c794a374c9fe6

> Ross Ulbricht à une date indéterminée (source : FreeRoss.org)

Reviewer: ```suggestion
Ross Ulbricht, à une date indéterminée (source : FreeRoss.org)
```

Après ses années universitaires, Ross est revenu à Austin, sa ville natale. N'ayant pas d'attrait particulier pour la voie scientifique, il exerce plusieurs emplois alimentaires. En 2010, il gère [Good Wagon Books](https://web.archive.org/web/20101107054013/http://goodwagon.com/), une entreprise de revente de livres d'occasion, qu'un ami lui a cédé. Il a sous sa charge quelques employés, mais le travail ne l'intéresse pas. Il est bien plus obsédé par son projet personnel.

Au cours de l'été 2010, Ross décide de cultiver des champignons hallucinogènes dans le but d'en faire le premier produit vendu sur son site web. Il loue pour cela un local à Bastrop, en périphérie d'Austin, se procure l'équipement nécessaire et, à l'instar de Walter White dans la série télévisée *Breaking Bad*, se transforme de simple amateur de chimie en producteur de drogue.

> Pour construire le site, il [demande](https://antilop.cc/sr/files/2015_01_22_Ulbricht_trial_transcript_W2_D3.pdf) de l'aide à Richard Bates, un ami d'université ayant travaillé pour PayPal et pour eBay. Ross ne lui dit pas intialement quelle est la nature de son projet, mais Richard accepte de lui donner quelques conseils en informatique. À la fin de l'année 2010, la plateforme Silk Road est donc prête à être lancée.

Reviewer: ```suggestion
Pour construire le site, il [demande](https://antilop.cc/sr/files/2015_01_22_Ulbricht_trial_transcript_W2_D3.pdf) de l'aide à Richard Bates, un ami d'université ayant travaillé pour PayPal et pour eBay. Ross ne lui dit pas initialement quelle est la nature de son projet, mais Richard accepte de lui donner quelques conseils en informatique. À la fin de l'année 2010, la plateforme Silk Road est donc prête à être lancée.
```

### Le lancement de la plateforme

> Après avoir mis la plateforme en place, Ross Ulbricht se consacre à en faire la promotion. Le 27 janvier 2011, il s'inscrit sous le pseudonyme altoid sur le forum de *The Shroomery*, un site consacré aux champignons hallucinogènes, et y [publie](https://www.shroomery.org/forums/showflat.php/Number/13860995) un message mentionnant la place de marché, dans laquel il prétend être tombé dessus par hasard. Le 29 janvier, il [réitère](https://bitcointalk.org/index.php?topic=175.msg42670#msg42670) cette manœuvre sur le forum de Bitcoin, au sein du fil consacré au « magasin d'héroïne ». Il écrit alors :

Reviewer: ```suggestion
Après avoir mis la plateforme en place, Ross Ulbricht se consacre à en faire la promotion. Le 27 janvier 2011, il s'inscrit sous le pseudonyme altoid sur le forum de *The Shroomery*, un site consacré aux champignons hallucinogènes, et y [publie](https://www.shroomery.org/forums/showflat.php/Number/13860995) un message mentionnant la place de marché, dans lequel il prétend être tombé dessus par hasard. Le 29 janvier, il [réitère](https://bitcointalk.org/index.php?topic=175.msg42670#msg42670) cette manœuvre sur le forum de Bitcoin, au sein du fil consacré au « magasin d'héroïne ». Il écrit alors :
```

>
> original: "The state is the primary source of violence, oppression, theft and all forms of coercion. \[...\] Stop funding the state with your tax dollars and direct your productive energies into the black market."

Cet article a pour effet de révéler l'existence de Silk Road au grand public, la nouvelle étant reprise par la presse généraliste, comme [*The Atlantic*](https://www.theatlantic.com/technology/archive/2011/06/libertarian-dream-a-site-where-you-buy-drugs-with-digital-dollars/239776/). Cette publicité inédite a pour conséquence de faire exploser le trafic de la plateforme. Les serveurs ont du mal à tenir la charge, à tel point que Ross [est contraint](https://bitcointalk.org/index.php?topic=3984.msg189007#msg189007) de geler les inscriptions.

On voit aussi des [concurrents](https://gwern.net/dnm-survival) émerger, copiant le modèle de Silk Road, comme Black Market Reloaded qui est lancé fin juin. Silk Road restera néanmoins numéro 1 jusqu'à sa fermeture, grâce à l'« avantage du précurseur » et à sa philosophie unique.

### Les réactions à la popularisation du site

> La publicité suscitée par l'article de Gawker ne plait pas à tout le monde. Le fait que Silk Road soit sous sous le feu des projecteurs a pour effet collatéral d'intensifier l'attention qui est portée à Bitcoin (ne serait-ce parce que les clients doivent se procurer du bitcoin pour acheter des produits). Et il s'avère qu'une partie de la communauté de Bitcoin est hostile à ce rapprochement entre Bitcoin et le trafic de drogue.

Reviewer: ```suggestion
La publicité suscitée par l'article de Gawker ne plait pas à tout le monde. Le fait que Silk Road soit sous le feu des projecteurs a pour effet collatéral d'intensifier l'attention qui est portée à Bitcoin (ne serait-ce parce que les clients doivent se procurer du bitcoin pour acheter des produits). Et il s'avère qu'une partie de la communauté de Bitcoin est hostile à ce rapprochement entre Bitcoin et le trafic de drogue.
```

> En particulier, le programmeur Jeff Garzik, qui s'investit alors dans le développement du logiciel, voit d'un mauvais œil cette association malvenue. Suite à la publication de l'article, il [contacte](https://www.theatlantic.com/technology/archive/2011/06/libertarian-dream-a-site-where-you-buy-drugs-with-digital-dollars/239776/) personnellement Adrien Chen par courriel pour lui expliquer que Bitcoin n'est pas aussi anonyme qu'il n'y parait, toutes les transactions étant enregistrées de façon permamente sur la chaîne. Il conclut son courriel par le jugement suivant :

Reviewer: ```suggestion
En particulier, le programmeur Jeff Garzik, qui s'investit alors dans le développement du logiciel, voit d'un mauvais œil cette association malvenue. Suite à la publication de l'article, il [contacte](https://www.theatlantic.com/technology/archive/2011/06/libertarian-dream-a-site-where-you-buy-drugs-with-digital-dollars/239776/) personnellement Adrien Chen par courriel pour lui expliquer que Bitcoin n'est pas aussi anonyme qu'il n'y parait, toutes les transactions étant enregistrées de façon permanente sur la chaîne. Il conclut son courriel par le jugement suivant :
```

C'est au cours de cette démarche que Ross commet une erreur fondamentale. Le 11 octobre, il publie une [annonce](https://bitcointalk.org/index.php?topic=47811.msg568744#msg568744) sur le forum de Bitcoin dans laquelle il déclare rechercher un « professionnel en informatique » pour une « startup bitcoin soutenue par capital-risque ». (original: "I'm looking for the best and brightest IT pro in the bitcoin community to be the lead developer in a venture backed bitcoin startup company.") Il utilise pour cela le pseudonyme altoid (avec lequel il a précédemment fait la promotion de Silk Road). Dans le message, il dévoile son adresse de courrier électronique, contenant son nom civil : `rossulbricht@gmail.com`. Cette erreur lui coûtera cher, car elle aidera l'enquête policière contre Silk Road à l'identifier et à l'arrêter deux ans plus tard.

### Un élément central de l'histoire de Bitcoin

> Ross Ulbricht a ainsi mis en œuvre une idée novatrice, basée sur ses convictions libertariennes et agoristes, se servant de la résistance à le censure de Bitcoin. Après une période de préparation de plus d'un an, il a lancé le prototype en janvier 2011. Les débuts ont été difficiles, mais ses efforts ont fini par payer : en juin 2011, Silk Road était connu du monde entier grâce à un article publié sur Gawker.

Reviewer: ```suggestion
Ross Ulbricht a ainsi mis en œuvre une idée novatrice, basée sur ses convictions libertariennes et agoristes, se servant de la résistance à la censure de Bitcoin. Après une période de préparation de plus d'un an, il a lancé le prototype en janvier 2011. Les débuts ont été difficiles, mais ses efforts ont fini par payer : en juin 2011, Silk Road était connu du monde entier grâce à un article publié sur Gawker.
```

La première plateforme de change d'envergure a été Mt. Gox, lancée « sur un coup de tête » en juillet 2010 par l'entrepreneur Jed McCaleb. Grâce à son fonctionnement automatisé, elle s'est rapidement imposée comme le moyen principal de changer des dollars en bitcoins, et inversement. Au début de l'année 2011, face à l'ampleur de la tâche, Jed McCaleb a décidé de céder cette plateforme à une personne plus qualifiée techniquement et davantage motivée : le développeur français Mark Karpelès.

La plateforme a joué un rôle central dans l'histoire de Bitcoin. Elle a été le lieu principal de la première réelle bulle spéculative sur le prix du bitcoin, qui a culminé à 32 $ en juin. Elle a aussi connu un piratage massif, et était liée à plusieurs incidents s'étant produit dans l'écosystème. Dans ce chapitre, nous parlerons des circonstances dans lesquelles s'est faite la reprise de Mt. Gox, du succès fulgurant qu'elle a rencontré à partir du printemps et des premiers problèmes qu'elle a subi.

### Mark Karpelès, le geek

> Mark Karpelès est né en France en 1984. Fils unique, il élevé par sa mère seule. Il s'intéresse rapidement pour les ordinateurs, les jeux vidéos et les mangas. Dans sa vingtaine, il est un geek [mal dans sa peau](https://web.archive.org/web/20140302234940/http://blog.magicaltux.net/2006/02/12/pensees-nocturnes/). En 2007, il apparait dans le documentaire *Suck my Geek*, consacré au mouvement geek français. En 2009, désireux de quitter la France où il ne s'est jamais réellement chez lui, il déménage au Japon [en juin 2009](https://web.archive.org/web/20090628035138/http://blog.magicaltux.net/2009/06/18/arrived-in-japan/) par attrait pour la culture locale. Là-bas, il fonde une société appelée Tibanne Co. Ltd. (comme son chat, Tibane), par le biais de laquelle il travaille en free-lance. Il gère notamment un service d'hébergement appelé [KalyHost](https://web.archive.org/web/20101231201212/https://www.kalyhost.com/).

Reviewer: ```suggestion
Mark Karpelès est né en France en 1984. Fils unique, il est élevé par sa mère seule. Il s'intéresse rapidement pour les ordinateurs, les jeux vidéos et les mangas. Dans sa vingtaine, il est un geek [mal dans sa peau](https://web.archive.org/web/20140302234940/http://blog.magicaltux.net/2006/02/12/pensees-nocturnes/). En 2007, il apparait dans le documentaire *Suck my Geek*, consacré au mouvement geek français. En 2009, désireux de quitter la France où il ne s'est jamais réellement chez lui, il déménage au Japon [en juin 2009](https://web.archive.org/web/20090628035138/http://blog.magicaltux.net/2009/06/18/arrived-in-japan/) par attrait pour la culture locale. Là-bas, il fonde une société appelée Tibanne Co. Ltd. (comme son chat, Tibane), par le biais de laquelle il travaille en free-lance. Il gère notamment un service d'hébergement appelé [KalyHost](https://web.archive.org/web/20101231201212/https://www.kalyhost.com/).
```

Reviewer: "s'intéresse pour les ordinateurs", ça sonne bizarre mais je ne suis pas sûr de moi ici. C'est pas "s'intéresser à" ?

Reviewer: "jeux vidéos" je pense qu'il n'y a pas de "s" à "vidéos"

> ![Mark Karpelès dans Suck My Geek en 2007](assets/en/mark-karpeles-suck-my-geek-2007.webp)

Reviewer: Normalement, on nomme les images séquentiellement sur 3 chiffres, en fonction de leur ordre d'apparition. Mais c'est quelque chose que l'on pourra modifier facilement à la fin avec un script.

Author: Je ne peux pas savoir à l'avance combien il y aura d'images ;)

Dès sa découverte, en bon passionné d'informatique, il se plonge dans les rouages de Bitcoin et commence à écrire des programmes en tirant parti. En novembre, il [commence](https://bitcointalk.org/index.php?topic=30.msg20699#msg20699) à accepter les paiements en bitcoins avec KalyHost (Ross Ulbricht [utilisera](https://cdn.arstechnica.net/wp-content/uploads/2015/01/Govt.motion.1.19.pdf) d'ailleurs ce service pour héberger la page web silkroadmarket.org). En décembre 2010, il [crée](https://bitcointalk.org/index.php?topic=2321.msg30872#msg30872) un nouveau wiki, Bitcoin.it, qui deviendra rapidement le wiki principal de Bitcoin. Il se met également en tête de développer une implémentation logicielle ([QBitcoin](https://web.archive.org/web/20110326023018/http://bitcoinweekly.com/articles/interview-with-magicaltux-on-qbitcoin)), sans pour autant aller au bout.

### L'achat de Mt. Gox

> Au début de l'année 2011, Jed McCaleb gère Mt. Gox tant bien que mal. La plateforme n'est pas sûre du tout, opinion partagée par certains membres de la communauté, comme [Mike Caldwell](https://bitcointalk.org/index.php?topic=4187.msg66477#msg66477). Et Jed n'est pas sûr de la conformité règlementaire de son l'activité. L'article de PC World a déjà eu un effet sur le prix, et il craint (à raison) que la tendance haussière ne se poursuive. Il se met donc à chercher un repreneur.

Reviewer: ```suggestion
Au début de l'année 2011, Jed McCaleb gère Mt. Gox tant bien que mal. La plateforme n'est pas sûre du tout, opinion partagée par certains membres de la communauté, comme [Mike Caldwell](https://bitcointalk.org/index.php?topic=4187.msg66477#msg66477). Et Jed n'est pas sûr de la conformité règlementaire de son activité. L'article de PC World a déjà eu un effet sur le prix, et il craint (à raison) que la tendance haussière ne se poursuive. Il se met donc à chercher un repreneur.
```

<!-- Britcoin/Intersango -->

Une deuxième plateforme est Britcoin, qui est [lancée](https://bitcointalk.org/index.php?topic=4984.msg73066#msg73066) le 26 mars par Amir Taaki (genjix), le jeune développeur britannique d'origine iranienne qui voulait envoyer un courrier à WikiLeaks en novembre 2010 pour recommander Bitcoin. Comme son nom l'indique, la plateforme est installé au Royaume-Uni et intègre le change entre la livre sterling et le bitcoin. À l'instar de David François, il [publie](https://bitcointalk.org/index.php?topic=4579.msg67047#msg67047) le logiciel sous-jacent [sous licence libre](https://github.com/dooglus/intersango/blob/master/LICENSE) (AGPL), qu'il baptise « Intersango » (de l'espéranto *interŝanĝo* signifiant échange). Amir est rapidement [rejoint](https://web.archive.org/web/20120129171120/https://britcoin.co.uk/about-us.php) par Donald Norman (Donald_Norman) et Patrick Strateman (phantomcircuit) pour gérer la plateforme.

> ![Patrick Strateman en 2016](assets/en/patrick-strateman-phantomcircuit-2016.webp)

Reviewer: Certaines images comme celle-ci vont être trop grosses à l'écran, car la plateforme étend l'image au maximum dans la largeur. J'ai un script pour vérifier cela et mettre les images avec proportion verticale trop grande dans un cadre pour éviter qu'elles ressortent géantes. Il faudra penser à l'exécuter sur ta formation avant la publication.

Author: Même si l'image est petite ?

> Comme nous l'avons dit, Mt. Gox constitue un cible idéale. Elle est régulièrement la cible d'attaques par déni de service (par exemple le [1er mai](https://bitcointalk.org/index.php?topic=6931.msg101451#msg101451)). De plus, elle a déjà échappé au pire en mai, lorsqu'un pirate s'tait emparé de 300 000 BTC avant d'en rendre 297 000.

Reviewer: ```suggestion
Comme nous l'avons dit, Mt. Gox constitue une cible idéale. Elle est régulièrement la cible d'attaques par déni de service (par exemple le [1er mai](https://bitcointalk.org/index.php?topic=6931.msg101451#msg101451)). De plus, elle a déjà échappé au pire en mai, lorsqu'un pirate s'était emparé de 300 000 BTC avant d'en rendre 297 000.
```

<!-- Le 17 juin, le pirate [annonce](https://web.archive.org/web/20110619054549/http://pastebin.com/xhnNdvte) vendre la base de données de Mt. Gox sur Pastebin. Il signe son message du pseudonyme cRazIeStinGeR. -->

> Toutefois, le dimanche 19 juin, la réalité rattrappe Mark. Vers 17 heures, le pirate parvient à accéder au compte de Jed McCaleb, qui contient beaucoup de fonds. Puisque la limite de retrait journalière est de 1 000 $, il cherche à faire baisser le prix le plus possible afin de retirer un maximum de bitcoins, et se met donc à vendre les bitcoins sur le compte. Cette opération provoque un krach éclair (*flash crash*) : le prix, qui stationnait autour des 17,5 $ dans la journée, baisse progressivement puis chute jusqu'à 0,01 $ à 17:51 UTC. Le pirate réussit à dérober ainsi 2 000 BTC, soit 35 000 $ au cours en vigueur.

Reviewer: ```suggestion
Toutefois, le dimanche 19 juin, la réalité rattrape Mark. Vers 17 heures, le pirate parvient à accéder au compte de Jed McCaleb, qui contient beaucoup de fonds. Puisque la limite de retrait journalière est de 1 000 $, il cherche à faire baisser le prix le plus possible afin de retirer un maximum de bitcoins, et se met donc à vendre les bitcoins sur le compte. Cette opération provoque un krach éclair (*flash crash*) : le prix, qui stationnait autour des 17,5 $ dans la journée, baisse progressivement puis chute jusqu'à 0,01 $ à 17:51 UTC. Le pirate réussit à dérober ainsi 2 000 BTC, soit 35 000 $ au cours en vigueur.
```

> Le krach éclair n'est cependant pas le seul évènement dramatique qui se produit ce jour-là. Quelques heures plus tard, [vers 19 heures 30](https://news.ycombinator.com/item?id=2671612), le pirate [rend public](https://bitcointalk.org/index.php?topic=19576.msg244940#msg244940) la base de données des utilisateurs. Cela permettra à d'autres personnes malentionnées et plus expérimentées de complètement l'exploiter. Elles accèderont ainsi à des services comme [GMail](https://bitcointalk.org/index.php?topic=19641.msg245714#msg245714), [TradeHill](https://www.youtube.com/watch?v=8ygb9D9vnL4&t=253s) ou encore MyBitcoin...

Reviewer: ```suggestion
Le krach éclair n'est cependant pas le seul évènement dramatique qui se produit ce jour-là. Quelques heures plus tard, [vers 19 heures 30](https://news.ycombinator.com/item?id=2671612), le pirate [rend public](https://bitcointalk.org/index.php?topic=19576.msg244940#msg244940) la base de données des utilisateurs. Cela permettra à d'autres personnes malintentionnées et plus expérimentées de complètement l'exploiter. Elles accèderont ainsi à des services comme [GMail](https://bitcointalk.org/index.php?topic=19641.msg245714#msg245714), [TradeHill](https://www.youtube.com/watch?v=8ygb9D9vnL4&t=253s) ou encore MyBitcoin...
```

> « Les bitcoins ne sont pas sécurisés, comme le montrent à la fois le vol récent et le problème des mots de passe. Ils ne sont pas liquides, ni ne forment une réserve de valeur comme en témoigne l'effondrement du prix, et s'ils ne sont rien de tout cela, ils ne seront pas non plus un bon moyen d'échange, car qui voudrait les accepter ? »
>
> "Bitcoins aren't secure, as both the recent theft and this password problem show. They're not liquid, nor a store of value, as the price collapse shows and if they're none of those things then they'll not be a great medium of exchange either as who would want to accept them?"

> La 21 juin, afin d'améliorer la situation et de rassurer les clients, Mark Karpelès et Adam Turner [passent](https://www.youtube.com/watch?v=-0XvP841jaM) dans le *Bitcoin Show* de Bruce Wagner. Le 23, Mark [déplace](https://bitcointalk.org/index.php?topic=21436.msg268800#msg268800) 424 242 bitcoins sur la chaîne pour prouver la solvabilité de l'entreprise, et la plateforme rouvre finalement le 27. Mark a compris qu'il fallait mieux communiquer : en juillet, il [réalisera](https://www.youtube.com/watch?v=wG-Wt0dAm_U) même un épisode du *Bitcoin Show* en français avec David François !

Reviewer: ```suggestion
Le 21 juin, afin d'améliorer la situation et de rassurer les clients, Mark Karpelès et Adam Turner [passent](https://www.youtube.com/watch?v=-0XvP841jaM) dans le *Bitcoin Show* de Bruce Wagner. Le 23, Mark [déplace](https://bitcointalk.org/index.php?topic=21436.msg268800#msg268800) 424 242 bitcoins sur la chaîne pour prouver la solvabilité de l'entreprise, et la plateforme rouvre finalement le 27. Mark a compris qu'il fallait mieux communiquer : en juillet, il [réalisera](https://www.youtube.com/watch?v=wG-Wt0dAm_U) même un épisode du *Bitcoin Show* en français avec David François !
```

Toutefois, cette croissance fulgurante cachait une grande fragilité. La plateforme Mt. Gox a subi plusieurs piratages durant cette période, ce qui fait qu'elle a accumulé les pertes au fil des mois : plus d'une centaine de milliers de bitcoins manquaient à l'appel à la fin de l'été. Cela présageait une chute abrupte. Mais Mt. Gox allait d'abord devenir une plateforme énorme en l'espace de deux ans et demi.

## Bitcoin et l'activisme politique (mars–novembre 2011)

Bitcoin est un objet éminemment politique en ce qu'il est un outil de libération et qu'il permet de faire des choses hors du cadre légal. C'est pourquoi il a tout naturellement attiré les individus motivés idéologiquement, qui ont souvent été ses partisans les plus passionnés. Son essor intervient de plus dans une période de grand scepticisme et de colère vis-à-vis du système bancaire, suite à la crise financière de 2008.

> Bitcoin a en particulier bénéficié du mouvement libertarien américain, qui prônait à la fois la liberté économique (propre aux républicains) et une liberté des mœurs (propre aux démocrates) et constituait de ce fait une sorte de troisième voie politique aux États-Unis. Les libertariens sont ainsi devenus les plus grand promoteurs de la cryptomonnaie, à l'instar de Roger Ver, Erik Voorhees ou encore Jon Matonis. Ils ont également favorisé son développement économique dans l'État du New Hampshire, grâce à l'action du *Free State Project*.

Reviewer: ```suggestion
Bitcoin a en particulier bénéficié du mouvement libertarien américain, qui prônait à la fois la liberté économique (propre aux républicains) et une liberté des mœurs (propre aux démocrates) et constituait de ce fait une sorte de troisième voie politique aux États-Unis. Les libertariens sont ainsi devenus les plus grands promoteurs de la cryptomonnaie, à l'instar de Roger Ver, Erik Voorhees ou encore Jon Matonis. Ils ont également favorisé son développement économique dans l'État du New Hampshire, grâce à l'action du *Free State Project*.
```

### Bitcoin et le libertarianisme

> L'année 2011 est marquée par le début de la campagne de Ron Paul pour l'élection présidentielle de 2012. Ron Paul est homme politique libertarien, représentant du 14e district de l'État du Texas à la Chambre des Représentants. Il est surnommé « Doctor No » parce qu'il a coutume de voter contre tous les projets de loi qui réduisent le liberté des citoyens ou qui augmentent les dépenses publiques. Après avoir brigué l'investiture républicaine pour l'élection de 2008, il fait de même pour celle de 2012. Bien qu'elles aient peu de chances d'aboutir, ses candidatures enthousiasment un grand nombre de libertariens et [inspirent](https://reason.com/2012/05/11/welcome-to-ron-pauls-revolution-the-webs/) un véritable mouvement populaire. La seconde campagne de Ron Paul pour l'élection présidentielle débute en [mai 2011](https://web.archive.org/web/20110516130810/http://politicalticker.blogs.cnn.com/2011/05/13/breaking-rep-ron-paul-announces-third-bid-for-presidency/).

Reviewer: ```suggestion
L'année 2011 est marquée par le début de la campagne de Ron Paul pour l'élection présidentielle de 2012. Ron Paul est un homme politique libertarien, représentant du 14e district de l'État du Texas à la Chambre des Représentants. Il est surnommé « Doctor No » parce qu'il a coutume de voter contre tous les projets de loi qui réduisent la liberté des citoyens ou qui augmentent les dépenses publiques. Après avoir brigué l'investiture républicaine pour l'élection de 2008, il fait de même pour celle de 2012. Bien qu'elles aient peu de chances d'aboutir, ses candidatures enthousiasment un grand nombre de libertariens et [inspirent](https://reason.com/2012/05/11/welcome-to-ron-pauls-revolution-the-webs/) un véritable mouvement populaire. La seconde campagne de Ron Paul pour l'élection présidentielle débute en [mai 2011](https://web.archive.org/web/20110516130810/http://politicalticker.blogs.cnn.com/2011/05/13/breaking-rep-ron-paul-announces-third-bid-for-presidency/).
```

Certains libertariens essaient de s'opposer à ce rejet, bien qu'ils appartiennent au même mouvement et ont les mêmes références. C'est le cas du blogueur [Michael Suede](https://web.archive.org/web/20110609055336/http://www.libertariannews.org/2011/06/06/libertarian-goldbugs-hating-on-bitcoin-free-market-money/), de [Jon Matonis](https://themonetaryfuture.blogspot.com/2011/06/why-are-libertarians-against-bitcoin.html) ou encore du tout jeune [Vitalik Buterin](https://web.archive.org/web/20110617050611/http://bitcoinweekly.com/articles/bitcoin-and-the-goldbugs). D'autres activistes, moins sectaires, sont assez rapidement convaincus par Bitcoin, comme [Stefan Molyneux](https://odysee.com/@freedomain:b/stefan-molyneux-in-2011-bitcoins-digital:e) ou [Adam Kokesh](https://web.archive.org/web/20120517174507/https://bitcoinmagazine.net/adam-kokesh-on-bitcoin-and-free-market-money).

### L'émission de radio Free Talk Live

> Parmi les premières personnes à s'intéresser à Bitcoin dans les sphères libertariennes, on trouve les animateurs de Free Talk Live, Ian Freeman et Mark Edge. Free Talk Live est une émission de radio libre libertarienne existant depuis 2002 et enregistrée alors depuis la ville de Keene dans le New Hampshire. Elle est diffusée dans l'ensemble des États-Unis par le biais du réseau Genesis Communications Networ (le même réseau qu'Alex Jones). Elle est également disponible sous forme de podcast sur Internet.

Reviewer: ```suggestion
Parmi les premières personnes à s'intéresser à Bitcoin dans les sphères libertariennes, on trouve les animateurs de Free Talk Live, Ian Freeman et Mark Edge. Free Talk Live est une émission de radio libre libertarienne existant depuis 2002 et enregistrée alors depuis la ville de Keene dans le New Hampshire. Elle est diffusée dans l'ensemble des États-Unis par le biais du réseau Genesis Communications Network (le même réseau qu'Alex Jones). Elle est également disponible sous forme de podcast sur Internet.
```

Les deux animateurs sont enthousiasmés par Bitcoin. En particulier, la place de marché Silk Road [éveille](https://bitcointalk.org/index.php?topic=3984.msg66567#msg66567) aussi la curiosité de Ian Freeman. Quelques mois plus tard, ce dernier [écrira](https://bitcointalk.org/index.php?topic=12895.msg178537#msg178537) sur le forum de Bitcoin : « remercions Silk Road de m'avoir donné une raison de saisir ne serait-ce qu'une partie de la vision de Bitcoin ». (original: "thanks be to the Silk Road for giving me reason to catch the at least a portion of the vision of Bitcoin") Le 16 mars, la moitié de l'émission de Free Talk Live est [consacrée](http://traffic.libsyn.com/ftl/FTL2011-03-16.mp3) à Bitcoin et à Silk Road, ce qui ne manque pas d'attirer l'attention d'un public libertarien.

### Roger Ver

> L'un des auditeurs les plus fidèles de Free Talk Live est Roger Ver, et les mentions de Bitcoin ne tombent pas dans l'oreille d'un sourd. Roger Ver est un homme d'affaires américain, qui a fondé une société de revente de composants informatiques en ligne appelée MemoryDealers à la fin des années 90, et est devenu millionnaire par ce biais au bout de quelques années. En parallèle, il a développé des convictions libertariennes, notamment en lisant Ludwig von Mises, Murray Rothbard et Henry Hazlitt. Il entretient également une vision optimiste du le progrès technique, ayant été influencé par le futurologue transhumaniste Ray Kurzweil.

Reviewer: ```suggestion
L'un des auditeurs les plus fidèles de Free Talk Live est Roger Ver, et les mentions de Bitcoin ne tombent pas dans l'oreille d'un sourd. Roger Ver est un homme d'affaires américain, qui a fondé une société de revente de composants informatiques en ligne appelée MemoryDealers à la fin des années 90, et est devenu millionnaire par ce biais au bout de quelques années. En parallèle, il a développé des convictions libertariennes, notamment en lisant Ludwig von Mises, Murray Rothbard et Henry Hazlitt. Il entretient également une vision optimiste du progrès technique, ayant été influencé par le futurologue transhumaniste Ray Kurzweil.
```

> « J'ai réalisé qu'il s'agissait de la meilleure forme de monnaie que le monde entier ait jamais connue. J'étais réellement surexcité parce que je sentais que la vocation de ma vie était arrivée sous la forme de Bitcoin. Je ne suis pas sorti de chez moi pendant plus d'une semaine et j'ai passé chaque instant à lire tout ce que je pouvais trouver sur Bitcoin. Le manque de sommeil m'a rendu tellement malade que mon amie a dû m'emmener à l'hôpital. »
>
> original: "I realised it's the best form of money that the entire world has ever seen. I literally got so excited because I felt my life's calling had arrived in the form of Bitcoin. I didn't leave my house for over a week and spent every waking moment reading every single thing I could find on Bitcoin. I got so sick from the lack of sleep that my friend had to take me to the hospital."

> Suite à cette expérience, Roger Ver s'investit dans la communauté. Le 26 avril, il [commence](https://bitcointalk.org/index.php?topic=4667.msg95746#msg95746) commence à accepter les paiements en bitcoins avec sa société MemoryDealers. Le 4 juin, il utilise un panneau d'affichage dont il dispose pour faire la promotion de Bitcoin et de son entreprise, MemoryDealers. Ce panneau est visible depuis une autoroute très fréquentée de Sunnyvale, dans la Silicon Valley, ce qui a pour conséquence d'[attirer](https://www.instagram.com/p/FNIFl/) l'attention sur Bitcoin dans le mouvement de bulle spéculative qui a alors lieu.

Reviewer: ```suggestion
Suite à cette expérience, Roger Ver s'investit dans la communauté. Le 26 avril, il [commence](https://bitcointalk.org/index.php?topic=4667.msg95746#msg95746) à accepter les paiements en bitcoins avec sa société MemoryDealers. Le 4 juin, il utilise un panneau d'affichage dont il dispose pour faire la promotion de Bitcoin et de son entreprise, MemoryDealers. Ce panneau est visible depuis une autoroute très fréquentée de Sunnyvale, dans la Silicon Valley, ce qui a pour conséquence d'[attirer](https://www.instagram.com/p/FNIFl/) l'attention sur Bitcoin dans le mouvement de bulle spéculative qui a alors lieu.
```

![Ira Miller en novembre 2011](assets/en/ira-miller-bitcoin-show-2011.webp)

Ira Miller en novembre 2011 (source : [Bitcoin Show](https://www.youtube.com/watch?v=VuLAhZsFc1A) sur Youtube)

Avec Bitcoin, Ira Miller souhaite « construire des systèmes alternatifs, pour concurrencer le statu quo effroyablement corrompu » (original: "to build alternate systems, to compete with the horrifically corrupt status quo"). En 2011, il joint le geste à la parole. En mai, il lance [BitMunchies](https://bitcointalk.org/index.php?topic=7078.msg103566#msg103566), un magasin en ligne spécialisé dans la nourriture (*the munchies* est un terme d'argot signifiant « la dalle » en anglais). Pendant l'été, il développe aussi [BTCinch](https://bitcointalk.org/index.php?topic=40088.msg488726#msg488726), un processeur de paiement permettant aux commerçants de convertir leurs bitcoins en dollars et inversement.

> Les deux hommes s'entendent très bien et leurs compétences sont complémentaires. Erik Voorhees cherche un programmeur pour développer une idée qu'il a eu et Ira accepte. Il s'agit de FeedZeBirds (littéralement « nourrissez les oiseaux »), une plateforme publicitaire permettant de payer pour mettre en avant un contenu sur Twitter en achetant des retweets, qu'ils [lancent](https://bitcointalk.org/index.php?topic=52638.msg628165#msg628165) le 23 novembre. C'est une utilisation limite (Twitter n'encourage certainement pas ce type d'exploitation) profitant de la résistance à la censure de Bitcoin et de l'absence de frais de transaction. Le 28 novembre, les deux associés [passent](https://www.youtube.com/watch?v=VuLAhZsFc1A) dans le *Bitcoin Show* pour promouvoir leur produit. Par la suite, ils développeront également le processeur de paiement Paysius et le service d'envoi de fonds par e-mail Coinapult.

Reviewer: ```suggestion
Les deux hommes s'entendent très bien et leurs compétences sont complémentaires. Erik Voorhees cherche un programmeur pour développer une idée qu'il a eue et Ira accepte. Il s'agit de FeedZeBirds (littéralement « nourrissez les oiseaux »), une plateforme publicitaire permettant de payer pour mettre en avant un contenu sur Twitter en achetant des retweets, qu'ils [lancent](https://bitcointalk.org/index.php?topic=52638.msg628165#msg628165) le 23 novembre. C'est une utilisation limite (Twitter n'encourage certainement pas ce type d'exploitation) profitant de la résistance à la censure de Bitcoin et de l'absence de frais de transaction. Le 28 novembre, les deux associés [passent](https://www.youtube.com/watch?v=VuLAhZsFc1A) dans le *Bitcoin Show* pour promouvoir leur produit. Par la suite, ils développeront également le processeur de paiement Paysius et le service d'envoi de fonds par e-mail Coinapult.
```

### Le Free State Project

> Comme on l'a laissé supposer, ces personnes ont des liens plus moins proches avec le Free State Project. Le Free State Project est un mouvement de migration politique, visant à regrouper les libertariens dans un État afin d'y influencer le gouvernement local. Il a été [initié](https://web.archive.org/web/20010727113202/http://www.webleyweb.com/tle/libe131-20010723-03.html) en juillet 2001 par Jason Sorens, doctorant à Yale, qui déplorait l'échec des efforts libertariens aux élections et qui invitait « les partisans de la liberté de tous bords » à s'intaller dans un petit État. En 2003, l'État du New Hampshire a été choisi en raison de sa \[culture\], sa devise officielle étant « *Live Free or Die* » (du français « Vivre libre ou mourir »). Les gens devaient signer une déclaration d'intention, pour un objectif de 20 000 personnes (qui sera atteint en 2016). Entretemps, les plus motivés ont déjà déménagé.

Reviewer: ```suggestion
Comme on l'a laissé supposer, ces personnes ont des liens plus ou moins proches avec le Free State Project. Le Free State Project est un mouvement de migration politique, visant à regrouper les libertariens dans un État afin d'y influencer le gouvernement local. Il a été [initié](https://web.archive.org/web/20010727113202/http://www.webleyweb.com/tle/libe131-20010723-03.html) en juillet 2001 par Jason Sorens, doctorant à Yale, qui déplorait l'échec des efforts libertariens aux élections et qui invitait « les partisans de la liberté de tous bords » à s'installer dans un petit État. En 2003, l'État du New Hampshire a été choisi en raison de sa \[culture\], sa devise officielle étant « *Live Free or Die* » (du français « Vivre libre ou mourir »). Les gens devaient signer une déclaration d'intention, pour un objectif de 20 000 personnes (qui sera atteint en 2016). Entretemps, les plus motivés ont déjà déménagé.
```

![Panneau de bienvenue à l'entrée du New Hampshire en 2018](assets/en/new-hampshire-welcoming-sign-2018.webp)

Panneau de bienvenue à l'entrée du New Hampshire en 2018 (source : [New Hampshire Public Radio](https://www.nhpr.org/the-exchange/2018-02-28/its-tax-season-how-does-new-hampshires-complex-unique-tax-system-work))

> En 2011, le New Hampshire est déjà devenu un haut-lieu du libertarianisme. Deux évènements y sont organisés tous les ans par le Free State Project. Le premier est le *New Hampshire Liberty Forum*, qui est une convention classique où interviennent de grands noms du libertarianisme et qui a lieu au cours de ces années-là à l'hôtel Crowne Plaza de Nashua, au sud de l'État. Le second est le *Porcupine Freedom Festival* (plus communément appelé « PorcFest »), qui est un festival d'une semaine se déroulant en été sur un terrain de camping près de Lancaster, plus au nord. L'ambiance y est plus décontractée, à tel point que l'évènement a été [qualifié](https://aaeblog.com/2011/06/way-long-gone-part-2/) en 2011 de « Woodstock pour les gens rationnels » par l'auteur libertarien Roderick T. Long (original: "Woodstock for rational people"). L'édition de cette année-là, qui se déroule en juin, est un succès et [réunit](https://mises.org/mises-daily/porcfest-2011) plus de 1 000 personnes.

Reviewer: ```suggestion
En 2011, le New Hampshire est déjà devenu un haut lieu du libertarianisme. Deux évènements y sont organisés tous les ans par le Free State Project. Le premier est le *New Hampshire Liberty Forum*, qui est une convention classique où interviennent de grands noms du libertarianisme et qui a lieu au cours de ces années-là à l'hôtel Crowne Plaza de Nashua, au sud de l'État. Le second est le *Porcupine Freedom Festival* (plus communément appelé « PorcFest »), qui est un festival d'une semaine se déroulant en été sur un terrain de camping près de Lancaster, plus au nord. L'ambiance y est plus décontractée, à tel point que l'évènement a été [qualifié](https://aaeblog.com/2011/06/way-long-gone-part-2/) en 2011 de « Woodstock pour les gens rationnels » par l'auteur libertarien Roderick T. Long (original: "Woodstock for rational people"). L'édition de cette année-là, qui se déroule en juin, est un succès et [réunit](https://mises.org/mises-daily/porcfest-2011) plus de 1 000 personnes.
```

Pas sûr de moi. Je crois qu'il n'y a pas de trait d'union.

Le New Hampshire, grâce à cette concentration de libertariens, deviendra au fil des années un lieu d'adoption de la cryptomonnaie. Outre sa mutation institutionnelle, il [se transformera](https://www.forbes.com/sites/kashmirhill/2014/06/12/the-free-state-project-a-libertarian-testing-ground-for-bitcoin-3d-printers-and-drones/) en véritable « terrain d'essai » pour Bitcoin et d'autres innovations techniques favorisant la liberté comme les imprimantes 3D.

### Occupy Wall Street

Le libertarianisme n'est cependant pas le seul mouvement politique à présenter des affinités avec Bitcoin. De l'autre côté de l'échiquier politique (si l'on considère que les positions économiques du libertarianisme sont plutôt situées à droite), il y a tout le mouvement anticapitaliste et antifinance qui a tout intérêt à s'intéresser à la cryptomonnaie.

> Toutefois, tout comme les libertariens, cette famille politique a tendance à rejeter Bitcoin, car il n'est pas conforme à leur vision de la monnaie. Il est trop rigide, trop peu souvent à la politique. Bitcoin n'est pas une monnaie sociale, pouvant être gérée de manière démocratique, telle que les militants l'envisagent au travers des monnaies locales. \[source ? p2p foundation ?\] Yánis Varoufákis, marxiste libertaire et futur ministre des finances de la Grèce, lui [reprochera](https://www.yanisvaroufakis.eu/2013/04/22/bitcoin-and-the-dangerous-fantasy-of-apolitical-money/) son caractère déflationniste et inégalitaire en 2013, en écrivant notamment qu'« il ne peut y avoir de monnaie dépolitisée capable d'"alimenter" une société industrielle avancée » (original: "there can be no de-politicised currency capable of 'powering' an advanced, industrial society").

Reviewer: Pas sûr de la formulation "trop peu souvent à la politique"

Author: soumis je crois

### L'importance du politique

Le politique constitue un élément essentiel de l'essor de Bitcoin entre 2011 et 2014. C'est en particulier grâce au mouvement libertarien, en dépit de ses dissensions internes, que Bitcoin a pu prospérer, que ce soit par le biais d'individus comme Roger Ver ou Erik Voorhees, ou par l'intermédiaire du *Free State Project*. Plus à gauche, le mouvement *Occupy Wall Street* a également joué un rôle dans l'essor du bitcoin comme monnaie de désobéissance.

Pourtant, cet aspect idéologique était en opposition avec une force autrement plus attractive : celle de l'intérêt financier. Une tension s'est ainsi construite au cours de ces années entre les deux aspects. Mais avant cela, il fallait construire les outils permettant aux acteurs financiers de venir s'investir dans l'écosystème...

## L'éclatement de la grande bulle de 2011 (août 2011–juin 2012)

> En juin 2011, l'engouement spéculatif de juin a été stoppé net par la piratage de Mt. Gox et par sa mise hors ligne pendant des jours. Le fiasco lié à la fermeture de MyBitcoin à la fin du mois de juillet a finalement provoqué le dégonflement de la bulle. Dans les jours qui ont suivi le prix a baissé en flèche jusqu'à 6 $, et a finit par tomber à 2 $ en novembre.

Reviewer: ```suggestion
En juin 2011, l'engouement spéculatif de juin a été stoppé net par le piratage de Mt. Gox et par sa mise hors ligne pendant des jours. Le fiasco lié à la fermeture de MyBitcoin à la fin du mois de juillet a finalement provoqué le dégonflement de la bulle. Dans les jours qui ont suivi le prix a baissé en flèche jusqu'à 6 $, et a fini par tomber à 2 $ en novembre.
```

> D'autres critiques émergent entretemps, plutôt au niveau macroéconomique. En septembre, James Surowiecki, journaliste américain connu pour avoir publié *La Sagesse des foules* (original: "The Wisdom of Crowds") en 2004, [publie](https://www.technologyreview.com/2011/08/23/191860/cryptocurrency/) un réquisitoire contre Bitcoin dans la *MIT Technology Review*. Il accuse la limite du nombres de bitcoins d'encourager à la thésaurisation, ce qui l'empêcherait de devenir un intermédiaire d'échange, créant en dernier lieu une situation où « la grande majorité des bitcoins sont détenus par des personnes qui espèrent les vendre à d'autres personnes » (original: "the vast majority of bitcoins are held by people hoping to sell them to other people") et où . Le 7 septembre, Paul Krugman, économiste néokeynésien et lauréat du prix Nobel d'économie en 2008, [reprend](https://archive.nytimes.com/krugman.blogs.nytimes.com/2011/09/07/golden-cyberfetters/) ce raisonnement dans un article publié sur le site *New York Times*, intitulé « Golden Cyberfetters » (« Cyberchaînes dorées » en français), en référence à l'ouvrage *Golden Fetters* de Barry Eichengreen, soutenant que la grande dépression des années 1930 a été la conséquence du système trop restrictif de l'étalon-or. Il argue que « Bitcoin a créé son propre modèle privé d'étalon-or, dans lequel la masse monétaire est fixe »  (original: "Bitcoin has created its own private gold standard world, in which the money supply is fixed rather than subject to increase via the printing press.") et que la déflation résultante a fait que « la valeur réelle des transactions en bitcoins a diminué au lieu d'augmenter » et « le produit brut réel en bitcoins a fortement chuté » (original: "The actual value of transactions in Bitcoins has fallen rather than rising. In effect, real gross Bitcoin product has fallen sharply.").

Reviewer: ```suggestion
D'autres critiques émergent entretemps, plutôt au niveau macroéconomique. En septembre, James Surowiecki, journaliste américain connu pour avoir publié *La Sagesse des foules* (original: "The Wisdom of Crowds") en 2004, [publie](https://www.technologyreview.com/2011/08/23/191860/cryptocurrency/) un réquisitoire contre Bitcoin dans la *MIT Technology Review*. Il accuse la limite du nombre de bitcoins d'encourager à la thésaurisation, ce qui l'empêcherait de devenir un intermédiaire d'échange, créant en dernier lieu une situation où « la grande majorité des bitcoins sont détenus par des personnes qui espèrent les vendre à d'autres personnes » (original: "the vast majority of bitcoins are held by people hoping to sell them to other people") et où . Le 7 septembre, Paul Krugman, économiste néokeynésien et lauréat du prix Nobel d'économie en 2008, [reprend](https://archive.nytimes.com/krugman.blogs.nytimes.com/2011/09/07/golden-cyberfetters/) ce raisonnement dans un article publié sur le site *New York Times*, intitulé « Golden Cyberfetters » (« Cyberchaînes dorées » en français), en référence à l'ouvrage *Golden Fetters* de Barry Eichengreen, soutenant que la grande dépression des années 1930 a été la conséquence du système trop restrictif de l'étalon-or. Il argue que « Bitcoin a créé son propre modèle privé d'étalon-or, dans lequel la masse monétaire est fixe »  (original: "Bitcoin has created its own private gold standard world, in which the money supply is fixed rather than subject to increase via the printing press.") et que la déflation résultante a fait que « la valeur réelle des transactions en bitcoins a diminué au lieu d'augmenter » et « le produit brut réel en bitcoins a fortement chuté » (original: "The actual value of transactions in Bitcoins has fallen rather than rising. In effect, real gross Bitcoin product has fallen sharply.").
```

Reviewer: "s" enlevé à "nombre"

Reviewer: "et où..." il manque un bout de phrase je pense, ou alors supprimer le "et où"

![Le premier article de Paul Krugman sur Bitcoin](assets/en/krugman-golden-cyberfetters-2011.webp)

Le premier article de Paul Krugman sur Bitcoin (source : [archive](https://web.archive.org/web/20110909021607/http://krugman.blogs.nytimes.com/2011/09/07/golden-cyberfetters/) du New York Times)

> Le 17 octobre, le prix du BTC tombe brutalement en-dessous des 3,2 $ enregistrant ainsi une baisse de plus de 90 % par rapport au sommet de la bulle, chose que ne manque pas de [rappeler](https://arstechnica.com/tech-policy/2011/10/bitcoin-implodes-down-more-than-90-percent-from-june-peak/) Timothy Lee le lendemain. Dans un article publié sur Arstechnica, il décrit les problèmes liés à l'utilisation de la cryptomonnaie, comme les piratages ou la volatilité. Il ne paie pas cher de la peau de Bitcoin, car « cinq mois de mauvaises nouvelles et de dépréciation ont causé de sérieux dommages » à sa réputation (original: "But the value of a currency is built on its reputation, and five months of bad news and depreciation have done serious damage."). Dans la même veine, un article de *The Economist* ayant pour [titre](https://web.archive.org/web/20181019113320/https://www.economist.com/babbage/2011/10/21/the-bursting-of-the-bitcoin-bubble) « L'éclatement de la bulle du bitcoin » (original: "The bursting of the Bitcoin bubble") est publié le 21 octobre.

Reviewer: ```suggestion
Le 17 octobre, le prix du BTC tombe brutalement en dessous des 3,2 $ enregistrant ainsi une baisse de plus de 90 % par rapport au sommet de la bulle, chose que ne manque pas de [rappeler](https://arstechnica.com/tech-policy/2011/10/bitcoin-implodes-down-more-than-90-percent-from-june-peak/) Timothy Lee le lendemain. Dans un article publié sur Arstechnica, il décrit les problèmes liés à l'utilisation de la cryptomonnaie, comme les piratages ou la volatilité. Il ne paie pas cher de la peau de Bitcoin, car « cinq mois de mauvaises nouvelles et de dépréciation ont causé de sérieux dommages » à sa réputation (original: "But the value of a currency is built on its reputation, and five months of bad news and depreciation have done serious damage."). Dans la même veine, un article de *The Economist* ayant pour [titre](https://web.archive.org/web/20181019113320/https://www.economist.com/babbage/2011/10/21/the-bursting-of-the-bitcoin-bubble) « L'éclatement de la bulle du bitcoin » (original: "The bursting of the Bitcoin bubble") est publié le 21 octobre.
```

Reviewer: suppression du tiret "en-dessous"

Peu à peu, le prix remonte pour dépasser à nouveau 4 $ fin décembre, si bien que la presse est obligée d'admettre que Bitcoin n'est pas mort. Le 21 décembre, Timothy Lee [écrit](https://arstechnica.com/tech-policy/2011/12/bitcoins-comeback-should-western-union-be-afraid/) que « la stabilité apparente de la monnaie au cours du mois dernier \[l'a\] incité à y jeter un second coup d'œil » (original: "the currency's apparent stability over the last month has inspired us to give it a second look"). En particulier, il est amené à réfléchir ce qui donne de la valeur au bitcoin, trouve cette valeur non pas tant dans sa qualité de « monnaie alternative » que comme « méta-monnaie » permettant le « transfert de richesse à faible coût et sans réglementation entre les nations » (original: "Bitcoin is not so much an alternative currency as a 'metacurrency' that allows low-cost and regulation-free transfer of wealth between nations"), en concurrence avec Western Union.

### BitInstant, la plaque tournante du change en bitcoins

> Les plateformes de change se multiplient. Outre celles lancées pendant la phase haussière de la bulle, on voit plusieurs autres platefromes émerger pendant l'été. C'est le cas de BTC-e, [lancée](https://bitcointalk.org/index.php?topic=29698.msg437237#msg437237) par les Russes Alexander Vinnik et Aleksandr Bilyuchenko le 7 août ; de Bitstamp, [mise en ligne](https://web.archive.org/web/20140430233930/https://bitcointalk.org/index.php?topic=20292.220) par les Slovènes Nejc Kodrič et Damijan Merlak le 19 août ; et de Crypto X Change, [inaugurée](https://bitcointalk.org/index.php?topic=51457.msg613057#msg613057) le 10 novembre par les Australiens Ken et Colin Armitt.

Reviewer: ```suggestion
Les plateformes de change se multiplient. Outre celles lancées pendant la phase haussière de la bulle, on voit plusieurs autres plateformes émerger pendant l'été. C'est le cas de BTC-e, [lancée](https://bitcointalk.org/index.php?topic=29698.msg437237#msg437237) par les Russes Alexander Vinnik et Aleksandr Bilyuchenko le 7 août ; de Bitstamp, [mise en ligne](https://web.archive.org/web/20140430233930/https://bitcointalk.org/index.php?topic=20292.220) par les Slovènes Nejc Kodrič et Damijan Merlak le 19 août ; et de Crypto X Change, [inaugurée](https://bitcointalk.org/index.php?topic=51457.msg613057#msg613057) le 10 novembre par les Australiens Ken et Colin Armitt.
```

Toutefois, la plateforme Mt. Gox reste largement la plus utilisée. Cela pose problème aux (nombreux) clients occidentaux, car la plateforme est installée au Japon, et les dépôts et les retraits demandent de faire des virements internationaux pouvant durer des jours. Et PayPal censure toute activité relative au change de bitcoins.

> C'est ce que problème que tente de résoudre un jeune développeur gallois de 23 ans, appelé Gareth Nelson (garethnelsonuk), avec un service qui prendra le nom de BitInstant. Gareth a découvert Bitcoin au cours du printemps. Après s'être [intéressé](https://bitcointalk.org/index.php?topic=8780.msg131564#msg131564) au minage, il a [rencontré](https://bitcointalk.org/index.php?topic=9198.msg132990#msg132990) lui-même la difficulté d'envoyer de l'argent vers Mt. Gox, et a réalisé un échange de gré à gré pour se procurer des bitcoins.

Reviewer: ```suggestion
C'est ce problème que tente de résoudre un jeune développeur gallois de 23 ans, appelé Gareth Nelson (garethnelsonuk), avec un service qui prendra le nom de BitInstant. Gareth a découvert Bitcoin au cours du printemps. Après s'être [intéressé](https://bitcointalk.org/index.php?topic=8780.msg131564#msg131564) au minage, il a [rencontré](https://bitcointalk.org/index.php?topic=9198.msg132990#msg132990) lui-même la difficulté d'envoyer de l'argent vers Mt. Gox, et a réalisé un échange de gré à gré pour se procurer des bitcoins.
```

![Gareth Nelson vers 2011–2012](assets/en/gareth-nelson-bitinstant-2011.webp)

Gareth Nelson vers 2011–2012 (source : [archive](https://web.archive.org/web/20130118073422/https://www.bitinstant.com/aboutus) de BitInstant.com)

> Le 13 juin vers 4 heures du matin (UTC), Gareth Nelson [publie](https://bitcointalk.org/index.php?topic=16134.msg210887#msg210887) sur le fourm une idée de service de « paiement rapide pour Mt. Gox ». Il s'agit de constituer un intermédiaire qui vire les fonds sur la plateforme de change, une fois que ceux-ci ont été reçus dans le pays d'origine, typiquement les États-Unis. Il écrit :

Reviewer: ```suggestion
Le 13 juin vers 4 heures du matin (UTC), Gareth Nelson [publie](https://bitcointalk.org/index.php?topic=16134.msg210887#msg210887) sur le forum une idée de service de « paiement rapide pour Mt. Gox ». Il s'agit de constituer un intermédiaire qui vire les fonds sur la plateforme de change, une fois que ceux-ci ont été reçus dans le pays d'origine, typiquement les États-Unis. Il écrit :
```

Il présente Bitcoinica comme une « plateforme de trading de bitcoins avancée » (original: "Advanced Bitcoin Trading Platform"), qui est « élégante », « professionnelle » et « révolutionnaire » (original: "Elegant. Professional. Revolutionary."). La plateforme ne gère pas de portefeuille, les bitcoins étant conservés sur un compte dédié sur Mt. Gox. Les dépôts et retraits se font en bitcoins, par des codes Mt. Gox échangeables ou par virement bancaire international.

![Interface de Bitcoinica en février 2012](assets/en/bitcoinica-screenshot-201202.webp)

Interface de Bitcoinica en février 2012 (source : [archive de Bitcoinica.com](https://web.archive.org/web/20120209171119/http://bitcoinica.com/))

Le succès est au rendez-vous dès le début. Durant les premières 24 heures, 3 724 bitcoins sont échangés par 290 utilisateurs différents, soit l'équivalent de 19 000 $. Le volume sur les 7 premiers jours [est](https://web.archive.org/web/20111019032441/http://blog.bitcoinica.com/) de 160 865 bitcoins, soit plus de 800 000 $, faisant de Bitcoinica la deuxième plateforme de trading en nombre de dollars échangés derrière Mt. Gox.

> La plateforme héberge notamment la spéculation : l'effet de levier permet de décupler les gains (et les pertes), ce qui convient aux gros joueurus qui souhaitent tout miser sur un mouvement de prix. Le succès spéculatif est tel que le fait de se faire liquider sur la plateforme acquiert sa propre [appellation](https://bitcointalk.org/index.php?topic=49445.msg588749#msg588749) : « get Zhou Tonged » (qu'on peut traduire par « se faire zhou-tonguer » en français). Cette expression [donne son nom](https://www.youtube.com/watch?v=kdvTkddp1F0) à la chaîne Youtube *Zhou Tonged*, qui est lancée en mars 2012 pour héberger des parodies musicales à propos de Bitcoin et du trading. La plateforme a également l'intérêt d'offrir de la vente à découverte et de permettre aux acteurs économiques (comme les commerçants détenant du bitcoin en réserve) de se couvrir contre une éventuelle baisse du cours.

Reviewer: ```suggestion
La plateforme héberge notamment la spéculation : l'effet de levier permet de décupler les gains (et les pertes), ce qui convient aux gros joueurs qui souhaitent tout miser sur un mouvement de prix. Le succès spéculatif est tel que le fait de se faire liquider sur la plateforme acquiert sa propre [appellation](https://bitcointalk.org/index.php?topic=49445.msg588749#msg588749) : « get Zhou Tonged » (qu'on peut traduire par « se faire zhou-tonguer » en français). Cette expression [donne son nom](https://www.youtube.com/watch?v=kdvTkddp1F0) à la chaîne Youtube *Zhou Tonged*, qui est lancée en mars 2012 pour héberger des parodies musicales à propos de Bitcoin et du trading. La plateforme a également l'intérêt d'offrir de la vente à découvert et de permettre aux acteurs économiques (comme les commerçants détenant du bitcoin en réserve) de se couvrir contre une éventuelle baisse du cours.
```

Début mars 2012, Bitcoinica [accepte](https://bitcointalk.org/index.php?topic=67521.msg785510#msg785510) plus d'une dizaine de méthodes de dépôt et de retrait, dont le bitcoin directement, Liberty Reserve, PerfectMoney, Pecunix, ainsi que les virements SEPA en euros. Elle cumule alors quasiment autant de volume que Mt. Gox.

### L'émission de titres financiers avec GLBSE

> L'absence de besoin d'autorisation de Bitcoin pousse les membres de la communauté à imaginer des utilisations allant plus loin que le simple échange monétaire et la spéculation. Ils envisagent ainsi de libéraliser l'investissement en fournissant un moyen d'émettre des actions et des obligations pour les sociétés de l'écosystème. Il s'agit de contourner des réglementations imposées notamment par la *Securities and Exchange Commission* aux États-Unis (l'équivalent américain de l'AMF), en [profitant](http://trilema.com/2012/the-reasons-why-bitcoin-securities-cant-be-regulated-by-the-sec/) du caractère novateur de Bitcoin et du flou juridique qui l'entoure. L'idée est de créer un modèle alternatif au système financier classique, qui pourrait s'autoentrenir dans la démarche agoriste.

Reviewer: Trait d'union à "auto-entretenir" ?

L'un des individus s'impliquant dans ce mouvement est James McCarthy, qui utilise le pseudonyme Nefario sur le forum de Bitcoin (en référence au personnage du même nom dans le film d'animation *Moi, moche et méchant*). Celui-ci est un Irlandais vivant en Chine, qui occupe un poste de professeur d'anglais dans une université. Il s'intéresse à l'école autrichienne et aux idées des cypherpunks, [déclarant](https://web.archive.org/web/20110607050421/http://bitcoinweekly.com/articles/interview-with-nefario-founder-of-bitcoin-global) lui-même être « en quelque sorte un cypherpunk » (original: "something of a cypherpunk"). Il a [découvert](https://bitcointalk.org/index.php?topic=2502.msg33800#msg33800) l'existence de Bitcoin au cours de l'année 2010 et en a miné plus d'un millier. Il s'est ensuite impliqué dans plusieurs projets, dont le projet de magazine en ligne Bitcoin Weekly.

![James McCarthy (Nefario) en février 2012](assets/en/nefario-tedxleeds-feb-2012.webp)

James McCarthy (Nefario) en février 2012 (source : [archive](https://web.archive.org/web/20121008033716/https://www.tedxleeds.com/nefario-james-mccarthy/) de TEDxLeeds)

> En avril 2011, James McCarthy [lance](https://bitcointalk.org/index.php?topic=6500.msg94943#msg94943) GLBSE (prononcé glib-zee), acronyme de *GLobal Bitcoin Stock Exchange*. Les utilisateurs passent par une interface en ligne de commande qui permet d'interagir avec le serveur de la plateforme. Le client est une version modifiée d'un logiciel développé par Amir Taaki. D'après le site web, la plateforme [permet](https://web.archive.org/web/20110427033244/http://glbse.com/) « d'émettre des actions pour lever des capitaux ; de verser des dividendes aux actionnaires ; de soumettre des résolutions aux actionnaires et d'obtenir leur vote ; d'émettre et de vendre des obligations ; d'emprunter et de prêter sur le marché ; d'effectuer et de recevoir des remboursements de prêts ; d'émettre des contrats à terme ; de méta-spéculer sur des sociétés non cotées (sur d'autres places boursières) ; et d'échanger tout ce qui précède avec d'autres tradeurs sur le marché » (original: "Issue shares to raise capital; Pay dividends to shareholders; Put resolutions to shareholders and get their vote; Issue and sell bonds; Borrow and lend to the market; Make and recieve loan repayments; Issue futures contracts; Meta-trade on non listed companies (on other exchanges); And trade all of the above with other traders on the market").

Reviewer: La faute d'orthographe provient de l'original ? "recieve" -> "receive"

Author: Oui, je vais rajouter un [sic].

Le 11 avril 2012, après une [levée de fonds](http://trilema.com/2012/sa-ne-jucam-de-a-investitiile-n-bitcoini/#comment-78745) recueillant plus de 3 000 BTC (par émission d'actions et d'obligations) et une phase bêta, la plateforme [devient](https://bitcointalk.org/index.php?topic=76239.msg845954#msg845954) une bourse en bitcoins permettant d'émettre diverses actions (dont celle de MPOE) et est renommée MPEx (pour « Mircea Popescu's Exchange »).

![Capture de MPEx, août 2012](assets/en/mpex-20120829.webp)

Capture de MPEx, août 2012 (source : [archive](https://web.archive.org/web/20120829092213/http://polimedia.us/bitcoin/mpex.php))

La plateforme rencontrera un succès certain \[lequel ?\]. Elle hébergera notamment l'« IPO » de SatoshiDICE \[dire ce que c'est\]. Elle évoluera avec le temps, [intégrant](http://trilema.com/2012/futures-on-mpex/) les contrats à terme en décembre.

### Une phase de construction

> L'éclatement de la grande bulle de 2011 à la fin de l'été a été un moment difficile pour la communauté de Bitcoin. Le prix en dollars a en effet connu une baisse de plus de 90 % par rapport à son sommet de juin. Toutefois, la période a été proprice à la construction. C'est ainsi que l'on a vu fleurir une multitude de services financiers qui ont contribué au succès ultérieur de Bitcoin : BitInstant, Bitcoinica, GLBSE et MPEx.

Reviewer: ```suggestion
L'éclatement de la grande bulle de 2011 à la fin de l'été a été un moment difficile pour la communauté de Bitcoin. Le prix en dollars a en effet connu une baisse de plus de 90 % par rapport à son sommet de juin. Toutefois, la période a été propice à la construction. C'est ainsi que l'on a vu fleurir une multitude de services financiers qui ont contribué au succès ultérieur de Bitcoin : BitInstant, Bitcoinica, GLBSE et MPEx.
```

![Logo de Libbitcoin en septembre 2013](assets/en/libbitcoin-logo-sept-2013.webp)

Logo de Libbitcoin en septembre 2013 (source : [archive](https://web.archive.org/web/20130913065137/http://libbitcoin.dyne.org/) de libbitcoin.dyne.org)

En octobre, le logiciel [parvient](https://bitcointalk.org/index.php?topic=30646.msg553961#msg553961) à faire une vérification complète de la chaine. Amir Taaki rédige également un « Zen de Libbitcoin », structuré sur le même modèle que le Zen de Python, un ensemble de principes influençant la conception du langage de programmation Python. Il publiera également un [manifeste](https://libbitcoin.dyne.org/libbitcoin-manifesto.pdf) (*The Libbitcoin Manifesto*) en septembre 2013.

### Les portefeuilles légers

> Le problème des implémentations complètes est qu'elles demandent beaucoup de ressources et sont peu pratiques à utiliser pour beaucoup de monde. De plus, on ne peut pas les utiliser sur mobile. C'est pourquoi les nouveaux utilisateurs et les usagers occasionnels se tournent vers la solution de facilité : les applications dépositaires. En novembre 2010, Kiba met en place une [prime](https://bitcointalk.org/index.php?topic=1812.msg22319#msg22319) collective pour le développement d'un client Bitcoin sur mobile. Le 29 juillet 2011, la fermeture de MyBitcoin confirme la nécessité d'une solution intermédiaire. C'est ce qui catalyse le développement des protefeuilles légers (*lightweight wallets*).

Reviewer: ```suggestion
Le problème des implémentations complètes est qu'elles demandent beaucoup de ressources et sont peu pratiques à utiliser pour beaucoup de monde. De plus, on ne peut pas les utiliser sur mobile. C'est pourquoi les nouveaux utilisateurs et les usagers occasionnels se tournent vers la solution de facilité : les applications dépositaires. En novembre 2010, Kiba met en place une [prime](https://bitcointalk.org/index.php?topic=1812.msg22319#msg22319) collective pour le développement d'un client Bitcoin sur mobile. Le 29 juillet 2011, la fermeture de MyBitcoin confirme la nécessité d'une solution intermédiaire. C'est ce qui catalyse le développement des portefeuilles légers (*lightweight wallets*).
```

> Le première idée est d'utiliser un nœud en tant que serveur distant, en lequel l'utilisateur a totalement confiance. En février 2011, Amir Taaki [lance](https://bitcointalk.org/index.php?topic=3451.msg48538#msg48538) un client logiciel permettant de gérer un portefeuille localement en se connectant à un nœud existant appelé Spesmilo. Il s'agit d'un client fonctionnant initialement sur Linux, mais qui est rapidement porté sur Windows. Le nom est une référence au spesmilo, ancienne unité monétaire internationale, proposée par le mathématicien suisse René de Saussure (le frère du linguiste, Ferdinand) en 1907, dans le cadre du projet espérantiste.

Reviewer: ```suggestion
La première idée est d'utiliser un nœud en tant que serveur distant, en lequel l'utilisateur a totalement confiance. En février 2011, Amir Taaki [lance](https://bitcointalk.org/index.php?topic=3451.msg48538#msg48538) un client logiciel permettant de gérer un portefeuille localement en se connectant à un nœud existant appelé Spesmilo. Il s'agit d'un client fonctionnant initialement sur Linux, mais qui est rapidement porté sur Windows. Le nom est une référence au spesmilo, ancienne unité monétaire internationale, proposée par le mathématicien suisse René de Saussure (le frère du linguiste, Ferdinand) en 1907, dans le cadre du projet espérantiste.
```

https://groups.google.com/g/bitcoinj/c/fqUw1AKsxWI/m/pl6rzUSmUH4J

![Andreas Schildbach à la conférence re:publica à Berlin en mai 2016](assets/en/andreas-schildbach-rpen-may-2016-twitter-holger-bleich.webp)

Andreas Schildbach à la conférence re:publica à Berlin en mai 2016 (source : Holger Bleich sur [Twitter](https://twitter.com/_robinhob/status/727167827132727300))

![Interface du Bitcoin Wallet for Android en juillet 2011](assets/en/bitcoin-wallet-android-july-2011.webp)

Interface du Bitcoin Wallet for Android en juillet 2011 (source : Brandon Iles pour [Wired](https://web.archive.org/web/20110710020745/https://www.wired.com/gadgetlab/2011/07/bitcoin-android-app/))

> Problème de la confidentialité. Téléchargement du bloc complet (ce qui était prévu par Satoshi à l'origine), mais problème de bande passante. Bloom filters (BIP 37, https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki), proposés par Mike Hearn et Matt Corallo pour atténuer le problème.

Reviewer: ```suggestion
Problème de la confidentialité. Téléchargement du bloc complet (ce qui était prévu par Satoshi à l'origine), mais problème de bande passante. Bloom filters ((BIP 37)[https://github.com/bitcoin/bips/blob/master/bip-0037.mediawiki]), proposés par Mike Hearn et Matt Corallo pour atténuer le problème.
```

### Les portefeuilles déterministes

Les portefeuilles déterministes (*deterministic wallets*) sont des portefeuilles qui intègrent la dérivation des clés : au lieu d'être générées individuellement de manière aléatoire comme cela est fait dans le logiciel principal, les clés privées sont dérivées de manière déterministe d'une information (appelée la graine) grâce à des fonctions cryptographiques. La connaissance de cette information permet à l'utilisateur de récupérer l'intégralité de ses fonds, ce qui simplifie grandement la sauvegarde du portefeuille. Il n'y a plus besoin d'enregistrer encore et encore le fichier `wallet.dat` !

> Les portefeuille déterministes sont largement issus des réflexions du développeur américain Gregory Maxwell. Partisan du logiciel libre, ce dernier est alors [contributeur](https://commons.wikimedia.org/wiki/User:Gmaxwell) pour Wikipedia depuis 2004 et travaille pour la fondation Xiph.org, où il participe au développement du codec audio Opus. En mai 2011, il commence à s'engager dans Bitcoin. Il [s'essaie](https://bitcointalk.org/index.php?topic=7171.msg119591#msg119591) au minage. Il écrit aussi beaucoup et devient rapidement un gros contributeur sur le forum et sur le canal IRC \#bitcoin-dev, où il utilise le pseudonyme Gmaxwell.

Reviewer: ```suggestion
Les portefeuilles déterministes sont largement issus des réflexions du développeur américain Gregory Maxwell. Partisan du logiciel libre, ce dernier est alors [contributeur](https://commons.wikimedia.org/wiki/User:Gmaxwell) pour Wikipedia depuis 2004 et travaille pour la fondation Xiph.org, où il participe au développement du codec audio Opus. En mai 2011, il commence à s'engager dans Bitcoin. Il [s'essaie](https://bitcointalk.org/index.php?topic=7171.msg119591#msg119591) au minage. Il écrit aussi beaucoup et devient rapidement un gros contributeur sur le forum et sur le canal IRC \#bitcoin-dev, où il utilise le pseudonyme Gmaxwell.
```

> « Bitcoin devrait vraiment permettre et recommander par défaut l'utilisation de portefeuilles déterministes.   La sécurité supplémentaire apportée par les portefeuilles pré-générés actuels est relativement faible compte tenu de la manière dont la plupart des gens utilisent bitcoin. Et le fardeau des pertes dues à des sauvegardes incomplètes et à la pression accrue pour maintenir un seul portefeuille opérationnel est énorme. »
>
> "Bitcoin really ought to offer and default to using deterministic wallets.   The additional security of the current pre-generated ones is fairly small considering how most people use bitcoin and the liability of harm due to insufficient backups and increased pressure to keep a single wallet online is enormous."

> Il envisage deux types de dérivation. L'une (type 1) est simple et génère les clés privées directement à partir de la graine. L'autre (type 2) est plus complexe, mais permet de dériver les adresses d'une clé publique maitresse obtenue depuis la graine, sans accéder à la graine en question. Cela d'avoir un logiciel de traitement de paiements qui puisse générer une adresse à chaque transaction, sans que les clés privées ne soient exposées.

Reviewer: ```suggestion
Il envisage deux types de dérivation. L'une (type 1) est simple et génère les clés privées directement à partir de la graine. L'autre (type 2) est plus complexe, mais permet de dériver les adresses d'une clé publique maitresse obtenue depuis la graine, sans accéder à la graine en question. Cela permet d'avoir un logiciel de traitement de paiements qui puisse générer une adresse à chaque transaction, sans que les clés privées ne soient exposées.
```

Un portefeuille papier peut également être généré avec un outil spécifique. Ce type d'outil voit ainsi le jour à partir de l'été 2011. Le principal d'entre eux est BitAddress, développé par Pointbiz, qui est [lancé](https://web.archive.org/web/20111102021042/https://bitcointalk.org/index.php?topic=43496.0). Il s'agit d'un outil codé en Javascript fonctionnant sur navigateur. L'interface invite l'utilisateur à créer de l'entropie pour optimiser la génération de nombres pseudo-aléatoires. La clé privée et l'adresse résultantes peuvent ensuite être imprimées pour être conservés en toute sécurité. Il peut bien évidemment (et c'est recommandé !) être utilisé sans connexion à Internet, en enregistrant la page HTML.

![Portefeuille papier généré avec BitAddress en septembre 2011](assets/en/bitaddress-paper-wallet-20110922.webp)

Portefeuille papier généré avec BitAddress en septembre 2011 (source : [capture](https://web.archive.org/web/20120519160806/https://bitcointalk.org/index.php?topic=40264.140) de BitAddress.org)

> Un autre outil qui apparait à ce moment-là est VanityGen, [créé](https://web.archive.org/web/20111104120932/https://bitcointalk.org/index.php?topic=25804.0) par Samr7 en juillet 2011. Il s'agit d'un logiciel permettant de créer une adresse personnalisée (*vanity address*), commençant par des caractères spécifiques, en testant un grand nombre de clés privées. Cela est utile pour identifier les adresses de donation des individus. Par exemple, l'adresse de Pointbiz pour sa maintenance de BitAdress est `1NiNja1bUmhSoTXozBRBEtR8LeF9TGbZBN`.

Reviewer: ```suggestion
Un autre outil qui apparait à ce moment-là est VanityGen, [créé](https://web.archive.org/web/20111104120932/https://bitcointalk.org/index.php?topic=25804.0) par Samr7 en juillet 2011. Il s'agit d'un logiciel permettant de créer une adresse personnalisée (*vanity address*), commençant par des caractères spécifiques, en testant un grand nombre de clés privées. Cela est utile pour identifier les adresses de donation des individus. Par exemple, l'adresse de Pointbiz pour sa maintenance de BitAddress est `1NiNja1bUmhSoTXozBRBEtR8LeF9TGbZBN`.
```

> Enfin, la méthode la plus efficace pour se protéger contre le vol est celle du « portefeuille mental » (original: "brain wallet"), qui consiste à mémoriser une phrase secrète donnant accès aux bitcoins stockés à l'adresse qui correpond. Par exemple, la phrase secrète peut être hachés par SHA-256 pour obtenir la clé privée, donnant accès aux fonds. Cette méthode est [proposée](https://bitcointalk.org/index.php?topic=35082.msg436289#msg436289) par Mike Caldwell (Casascius) en août 2011 et [mise en avant](https://www.forbes.com/sites/jonmatonis/2012/03/12/brainwallet-the-ultimate-in-mobile-money/) dans Forbes par Jon Matonis en mars 2012. Elle est néanmoins très peu sûre si l'entropie n'est pas suffisante. Et elle est évidemment sujette aux aléas de la mémoire humaine ; il s'agit davantage d'une démonstration pratique de la puissance de Bitcoin qu'une application pérenne.

Reviewer: ```suggestion
Enfin, la méthode la plus efficace pour se protéger contre le vol est celle du « portefeuille mental » (original: "brain wallet"), qui consiste à mémoriser une phrase secrète donnant accès aux bitcoins stockés à l'adresse qui correspond. Par exemple, la phrase secrète peut être hachée par SHA-256 pour obtenir la clé privée, donnant accès aux fonds. Cette méthode est [proposée](https://bitcointalk.org/index.php?topic=35082.msg436289#msg436289) par Mike Caldwell (Casascius) en août 2011 et [mise en avant](https://www.forbes.com/sites/jonmatonis/2012/03/12/brainwallet-the-ultimate-in-mobile-money/) dans Forbes par Jon Matonis en mars 2012. Elle est néanmoins très peu sûre si l'entropie n'est pas suffisante. Et elle est évidemment sujette aux aléas de la mémoire humaine ; il s'agit davantage d'une démonstration pratique de la puissance de Bitcoin qu'une application pérenne.
```

> https://en.bitcoin.it/w/index.php?title=Brainwallet&oldid=23015 https://bitcoinmagazine.com/technical/brain-wallets-the-what-and-the-how-1333845334 https://bitcointalk.org/index.php?topic=51397.msg846234#msg846234 https://web.archive.org/web/20120413041417/http://brainwallet.org/

Reviewer: J'imagine que c'est pour tes notes. Tu vas enlever ces liens ou les intégrer au texte par la suite ?

Author: Enlever les liens, c'est un oubli !

Le 19 août, il [partage](https://bitcointalk.org/index.php?topic=38048.msg467209#msg467209) publiquement son projet de fabriquer des « vrais bitcoins physiques ». Le 6 septembre, il commence à en fabriquer. Le 6 septembre, il [annonce](https://bitcointalk.org/index.php?topic=41892.msg509883#msg509883) mettre en vente les « bitcoins physiques de Casascius » (original: "Casascius Physical Bitcoins"), qu'on connaitre plus communément sous le nom de « Casascius coins ». La clé privée est recouverte d'un hologramme personnalisé, si bien qu'on ne peut pas la récupérer sans endommager la pièce. Les 8 premiers caractères de l'adresse publique de Bitcoin sont visibles à l'extérieur et pré-imprimés sur l'hologramme par le fabricant. Détail surprenant : les premières pièces présentent une faute typographique ; Casascius est orthographié Casacius.

![Les premiers bitcoins de Casascius en 2011](assets/en/casasciusphysicalbitcoins-2011.webp)

Les premiers bitcoins de Casascius en 2011 (source : [casascius.com](https://www.casascius.com/casasciusphysicalbitcoins.jpg))

> Initialement disponibles sous formes de pièces de 1 et de 25 BTC, les bitcoins de Casascius rapidement déclinés en des pièces d'autres valeur et en lingots, allant jusqu'à 1000 BTC ! Les bitcoins de Casasicius sont [vendus](https://web.archive.org/web/20111107191637/https://www.casascius.com/) sur le site personnel de Casascius, mais [aussi](https://web.archive.org/web/20111028171555/http://www.memorydealers.com/bieq.html) sur MemoryDealers (le commerce de Roger Ver) à partir d'octobre. Ils se [retrouveront](https://odysee.com/@adamkokesh:2/moment-in-monetary-history-bitcoin-by:9) également dans un distributeur automatique présent au PorcFest de l'année suivante en juin 2012. Ils ont ainsi un but pédagogique. Comme l'[expliquera](https://bitcointalk.org/index.php?topic=41892.msg3727984#msg3727984) Mike Caldwell en 2013 :

Reviewer: ```suggestion
Initialement disponibles sous forme de pièces de 1 et de 25 BTC, les bitcoins de Casascius rapidement déclinés en des pièces d'autres valeurs et en lingots, allant jusqu'à 1000 BTC ! Les bitcoins de Casascius sont [vendus](https://web.archive.org/web/20111107191637/https://www.casascius.com/) sur le site personnel de Casascius, mais [aussi](https://web.archive.org/web/20111028171555/http://www.memorydealers.com/bieq.html) sur MemoryDealers (le commerce de Roger Ver) à partir d'octobre. Ils se [retrouveront](https://odysee.com/@adamkokesh:2/moment-in-monetary-history-bitcoin-by:9) également dans un distributeur automatique présent au PorcFest de l'année suivante en juin 2012. Ils ont ainsi un but pédagogique. Comme l'[expliquera](https://bitcointalk.org/index.php?topic=41892.msg3727984#msg3727984) Mike Caldwell en 2013 :
```

Spécificités : voir ["An Inquiry into Money Laundering Tools in the Bitcoin Ecosystem"](https://maltemoeser.de/paper/money-laundering.pdf)

C'est aussi la période où la réflexion autour des manières décentralisées de faire s'intensifie. Deux idées émergent : les certificats aveugles en surcouche de Bitcoin (Open Transactions, [proposé](https://bitcointalk.org/index.php?topic=847.msg9976#msg9976) par fellowtraveller en août 2010, eCash basé sur Bitcoin) ; les services de mélange décentralisés, publiant des transactions brisant les liens entre les entrées et les sorties (idée [proposée](https://bitcointalk.org/index.php?topic=12751.msg315793#msg315793) par Hashcoin en juillet 2011). Cette dernière piste conduira à la [formalisation](https://bitcointalk.org/index.php?topic=279249.msg2983902#msg2983902) de CoinJoin par Gregory Maxwell en 2013. Mais c'est une toute autre histoire...

### Une période de progrès pour les portefeuilles

> La période 2011–2012 a été faste pour les diverses solutions pour utiliser Bitcoin. Outre les applications dépositaires, on a vu se développer les portefeuilles logiciels comme *Bitcoin Wallet for Android* et Electrum. les portefeuilles web comme Blockchain.info, les générateurs de portefeuilles papier comme BitAddress, et les bitcoins physiques comme les Bitbills et les Casascius Coins. Des service de mélange sont apparus pour offrir plus de confidentialité aux utilisateurs.

Reviewer: ```suggestion
La période 2011–2012 a été faste pour les diverses solutions pour utiliser Bitcoin. Outre les applications dépositaires, on a vu se développer les portefeuilles logiciels comme *Bitcoin Wallet for Android* et Electrum. les portefeuilles web comme Blockchain.info, les générateurs de portefeuilles papier comme BitAddress, et les bitcoins physiques comme les Bitbills et les Casascius Coins. Des services de mélange sont apparus pour offrir plus de confidentialité aux utilisateurs.
```

Dans le prochain chapitre, nous continuerons à traiter la question du développement logiciel, en évoquant l'évolution du logiciel principal. Nous nous concentrerons sur le conflit autour de Pay to Script Hash, une fonctionnalité qui a été ajoutée au protocole en avril 2012.

+ MultiBit ? https://bitcointalk.org/index.php?topic=43616.msg520542#msg520542

## La poursuite de la ruée vers l'or numérique

> Le Far West se caractérise aussi par le ruée vers l'or : la découverte d'or en Californie en 1848 avait engendré une grade migration d'aventuriers espérant faire fortune. Bitcoin ayant été associé au métal précieux dès l'origine, il est naturel qu'un tel phénomène se soit développé. C'est ainsi que l'activité du minage de bitcoins a attiré des individus passionnés qui ont, par leur intérêt pécuniaire, contribué à la rendre plus efficace au cours du temps.

Reviewer: ```suggestion
Le Far West se caractérise aussi par le ruée vers l'or : la découverte d'or en Californie en 1848 avait engendré une grande migration d'aventuriers espérant faire fortune. Bitcoin ayant été associé au métal précieux dès l'origine, il est naturel qu'un tel phénomène se soit développé. C'est ainsi que l'activité du minage de bitcoins a attiré des individus passionnés qui ont, par leur intérêt pécuniaire, contribué à la rendre plus efficace au cours du temps.
```

### Les coopératives de minage et leur mode de rémunération

Principales coopératives, innovations, comparaison ([3/1/2012](https://en.bitcoin.it/w/index.php?title=Comparison_of_mining_pools&oldid=21595))

L'année 2011 est l'année où le minage groupé se généralise : [de nombreuses coopératives de minage](https://en.bitcoin.it/w/index.php?title=Comparison_of_mining_pools&oldid=21595) sont en effet lancées durant cette année-là. Bitcoin.cz, la coopérative de Marek Palatinus (alias Slush) [ouverte](https://bitcointalk.org/index.php?topic=1976.msg30520#msg30520) en décembre 2010, a rencontré le succès en quelques jours, en recueillant près de 3,5 % de la puissance de calcul du réseau, et n'a par conséquent pas manqué de faire des émules. Au fur et à mesure, les mineurs deviennent des « hacheurs », c'est-à-dire des personnes louant leur puissance de hachage à un opérateur, qui s'occupe de choisir les spécificités des blocs et les transactions à inclure. Différentes méthodes de rémunération sont adoptées.

> La première coopérative concurrente à émerger en 2011 est BitPenny, qui a la particularité d'appliquer un modèle de récompense simple : le *Pay Per Share* (PPS), ou paiement par part en français. Il s'agit de rémunerer les hacheurs à chaque preuve de travail partielle calculée, selon son degré, de façon immédiate, à partir du solde de la coopérative. L'opérateur prend donc l'intégralité de la variance à sa charge, ce qui est risquée. La coopérative BitPenny est [lancée](https://bitcointalk.org/index.php?topic=3266.msg45809#msg45809) le 8 février, mais est forcée de fermer le 2 avril, à la suite de pertes trop importantes. Elle sera néanmoins [relancée](https://bitcointalk.org/index.php?topic=36371.msg447984#msg447984) en août, en appliquant un autre modèle plus complexe (*Capped Pay Per Share with Recent Backpay* ou CPPSRB).

Reviewer: ```suggestion
La première coopérative concurrente à émerger en 2011 est BitPenny, qui a la particularité d'appliquer un modèle de récompense simple : le *Pay Per Share* (PPS), ou paiement par part en français. Il s'agit de rémunérer les hacheurs à chaque preuve de travail partielle calculée, selon son degré, de façon immédiate, à partir du solde de la coopérative. L'opérateur prend donc l'intégralité de la variance à sa charge, ce qui est risqué. La coopérative BitPenny est [lancée](https://bitcointalk.org/index.php?topic=3266.msg45809#msg45809) le 8 février, mais est forcée de fermer le 2 avril, à la suite de pertes trop importantes. Elle sera néanmoins [relancée](https://bitcointalk.org/index.php?topic=36371.msg447984#msg447984) en août, en appliquant un autre modèle plus complexe (*Capped Pay Per Share with Recent Backpay* ou CPPSRB).
```

Le 27 février, une nouvelle coopérative est [ouverte](https://web.archive.org/web/20111127072845/https://bitcointalk.org/index.php?topic=3889.0) par un [Russe](https://buildingbitcoin.org/bitcoin-dev/log-2011-02-19.html#l-2646) se faisant appeler Tycho (qu'il orthographie \[Tycho\]). Il s'agit de Deepbit, qui a pour particularité de récompenser ses clients à chaque bloc trouvé, au pro rata des parts qu'ils ont trouvées depuis le dernier bloc. À ce modèle proportionnel, Tycho [ajoute](https://bitcointalk.org/index.php?topic=3889.msg55799#msg55799) rapidement le modèle *Pay Per Share* à son offre, mais prélève des frais supérieurs pour ce mode (10 % au lieu de 3 %).

https://web.archive.org/web/20111010192050/http://eligius.st/~luke-jr/samples/800MH-3/

> Le 15 février, le développeur Jeff Garzik [publie](https://bitcointalk.org/index.php?topic=3493.msg49110#msg49110) un outil appelé Pushpool, un logiciel au code source ouvert permettant de déployer sa propre coopérative. Ce outil sera [agrémenté](https://bitcointalk.org/index.php?topic=13164.msg181362#msg181362) d'une interface graphique appelée SimpleCoin en juillet. ("pushpool - open source pool software" https://bitcointalk.org/index.php?topic=8707.0, "Pushpool Frontend: Simplecoin Opensource PHP/MySQL" https://bitcointalk.org/index.php?topic=13164.0)

Reviewer: ```suggestion
Le 15 février, le développeur Jeff Garzik [publie](https://bitcointalk.org/index.php?topic=3493.msg49110#msg49110) un outil appelé Pushpool, un logiciel au code source ouvert permettant de déployer sa propre coopérative. Cet outil sera [agrémenté](https://bitcointalk.org/index.php?topic=13164.msg181362#msg181362) d'une interface graphique appelée SimpleCoin en juillet. ("pushpool - open source pool software" https://bitcointalk.org/index.php?topic=8707.0, "Pushpool Frontend: Simplecoin Opensource PHP/MySQL" https://bitcointalk.org/index.php?topic=13164.0)
```

21/7 : https://web.archive.org/web/20110721162638/http://www.bitcoinwatch.com/

Juillet 2011, 49 % : https://bitcointalk.org/index.php?topic=26656.msg334436#msg334436

Trois coopératives sortent du lot : DeepBit, BTC Guild, Bitcoin.cz Mining (alias Slush Pool). La coopérative dominante, DeepBit, est gérée par un Russe \[?\] utilisant le pseudonyme Tycho. Elle [approchait](https://web.archive.org/web/20110706215910/http://www.bitcoinwatch.com/) des 50 % du taux de hachage en juillet 2011 ; au début de l'année 2012, elle possède autour des 30 % de la puissance de calcul. Les deux autres naviguent autour de 10–15 %.

- [blockchain.info, 19/12](https://web.archive.org/web/20111219124912/http://blockchain.info/pools), 144 blocs : 24 %, 14 %, 2 %
- [blockchain.info, 20/1](https://web.archive.org/web/20120120152812/http://blockchain.info/pools), 144 blocs : 42 %, 13 %, 4 %
- [blockorigin.pfoe.be, 18/3](https://web.archive.org/web/20120318030111/http://blockorigin.pfoe.be/), 2 016 blocs : 33,63 %, 11,90 %, 12,05 %
- [blockchain.info, 7/4](https://web.archive.org/web/20120407204745/http://blockchain.info/pools), 144 blocs : 32 %, 15 %, 11 %

### Luke-Jr et Eligius

> Une cooopérative atypique est Eligius.

Reviewer: ```suggestion
Une coopérative atypique est Eligius.
```

La spécialisation du minage s'accroit encore plus avec le développement des FPGA et des premiers modèles d'ASIC.

### Ouverture

ASIC, Butterfly Labs, Avalon

Professionnalisation du minage

## La bataille pour Pay to Script Hash

Après le départ de Satoshi Nakamoto, le développement du logiciel a été pris en charge par la communauté, qui a dû se coordonner, sous la supervision de Gavin Andresen. Il est en particulier question d'améliorer la programmabilité de Bitcoin, celle-ci ayant été temporairement restreinte par l'instauration de règles de relai sur le réseau.

Au cours de l'hiver 2011–2012, une controverse a eu lieu autour de la mise en œuvre de la signature multipartite dans Bitcoin. Celle-ci concernait, entre autres, le schéma appelé « Pay to Script Hash » (P2SH). Pete Rizzo et Aaron van Wirdum [appeleraient](https://bitcoinmagazine.com/technical/the-battle-for-p2sh-the-untold-story-of-the-first-bitcoin-war) la controverse « la bataille pour P2SH » (original: "the Battle for P2SH"). Cette première discorde majeure a posé la question de la gouvernance : qui décide comment le protocole est modifié ?

### Le développement du logiciel

> Après la sortie de la version 0.4 du logiciel le 23 septembre, qui marque la relève prise par Gavin Andresen en tant que mainteneur principal, le développement se poursuit. En particulier, une nouvelle interface graphique pour le logiciel est développée. Il s'agit de Bitcoin-Qt, qui est [développée](https://bitcointalk.org/index.php?topic=15276.msg203205#msg203205) par Wladmir van der Laan à partir de juin. Elle se base sur le framework graphique Qt en lieu et place de l'outil wxWindows utilisé par Satoshi. Celle-ci est [intégrée](https://bitcointalk.org/index.php?topic=45742.msg545547#msg545547) à la branche principale du code le 26 septembre et [présente](https://bitcointalk.org/index.php?topic=52480.msg626275#msg626275) dans la version 0.5 du logiciel qui sort le 21 novembre. Le logiciel sera d'ailleurs appelé Bitcoin-Qt/bitcoind ou Bitcoin-Qt tout court à partir de ce moment-là (bitcoind étant le démon).

Reviewer: ```suggestion
Après la sortie de la version 0.4 du logiciel le 23 septembre, qui marque la relève prise par Gavin Andresen en tant que mainteneur principal, le développement se poursuit. En particulier, une nouvelle interface graphique pour le logiciel est développée. Il s'agit de Bitcoin-Qt, qui est [développée](https://bitcointalk.org/index.php?topic=15276.msg203205#msg203205) par Wladimir van der Laan à partir de juin. Elle se base sur le framework graphique Qt en lieu et place de l'outil wxWindows utilisé par Satoshi. Celle-ci est [intégrée](https://bitcointalk.org/index.php?topic=45742.msg545547#msg545547) à la branche principale du code le 26 septembre et [présente](https://bitcointalk.org/index.php?topic=52480.msg626275#msg626275) dans la version 0.5 du logiciel qui sort le 21 novembre. Le logiciel sera d'ailleurs appelé Bitcoin-Qt/bitcoind ou Bitcoin-Qt tout court à partir de ce moment-là (bitcoind étant le démon).
```

### La signature multipartite

La signature multipartite, ou la multisignature, est une façon de requérir plusieurs signatures pour une dépense. Elle est la pierre angulaire du déploiement de contrats impliquant plusieurs personnes. Elle est utile pour la sécurisation des fonds qui, comme on l'a vu, est une question très importante à l'époque. Elle est aussi utile pour mettre en place un dépôt fiduciaire, permettant de rendre les échanges économiques plus sûrs pour le vendeur. Ce type d'application, mentionnée dans le livre blanc, a une demande certaine : Gavin Andresen a ainsi [géré](https://bitcointalk.org/index.php?topic=2134.msg27797#msg27797) lui-même un service de dépôt fiduciaire appelé [ClearCoin](https://clearcoin.blogspot.com/2011/06/ill-be-closing-clearcoin-to-new.html) entre décembre 2010 et juin 2011.

> Le langage de script contient un code opération, `OP_CHECKMULTISIG`, qui permet de requérir M signatures parmi N participants pour autoriser une dépense, avec M ≤ N. Les clés publiques des éventuels signataires sont incrites dans le script de verrouillage de la sortie transactionnelle. N'importe quel sous-ensemble de M signatures permettent de déverrouiller la pièce.

Reviewer: ```suggestion
Le langage de script contient un code opération, `OP_CHECKMULTISIG`, qui permet de requérir M signatures parmi N participants pour autoriser une dépense, avec M ≤ N. Les clés publiques des éventuels signataires sont inscrites dans le script de verrouillage de la sortie transactionnelle. N'importe quel sous-ensemble de M signatures permettent de déverrouiller la pièce.
```

### OP\_EVAL : exécuter un script dans un script

> L'instruction `OP_EVAL` permet d'exécuter un script à l'intérieur d'un script. Il est exécuté à la fin du script de verrouillage pour évaluer la validité du script donné, dont le hachage est comparée à l'empreinte donnée. Cela permet d'exécuter n'importe quel script.

Reviewer: ```suggestion
L'instruction `OP_EVAL` permet d'exécuter un script à l'intérieur d'un script. Il est exécuté à la fin du script de verrouillage pour évaluer la validité du script donné, dont le hachage est comparé à l'empreinte donnée. Cela permet d'exécuter n'importe quel script.
```

> ![Fonctionnement technique de l'opérateur OP_EVAL](assets/en/bytecoin-opeval-20111002.webp)

Reviewer: ```suggestion

| Stack                                        | Script                                                                 | Description                                          |
| -------------------------------------------- | ---------------------------------------------------------------------- | ---------------------------------------------------- |
| Empty                                        | `<sig> <script> OP_DUP OP_HASH160 <scriptHash> OP_EQUALVERIFY OP_EVAL` | scriptSig and scriptPubKey are combined.             |
| `<sig> <script>`                             | `OP_DUP OP_HASH160 <scriptHash> OP_EQUALVERIFY OP_EVAL`                | Constants are added to the stack.                    |
| `<sig> <script> <script>`                    | `OP_HASH160 <scriptHash> OP_EQUALVERIFY OP_EVAL`                       | Top stack item is duplicated.                        |
| `<sig> <script> <scriptHashA>`               | `OP_EQUALVERIFY OP_EVAL`                                               | Top stack item is hashed.                            |
| `<sig> <script> <scriptHashA> <scriptHashA>` | `OP_EQUALVERIFY OP_EVAL`                                               | Constant added.                                      |
| `<sig> <script>`                             | `OP_EVAL`                                                              | Equality is checked between the top two stack items. |
| `<sig>`                                      | `<pubKey> OP_CHECKSIG`                                                 | Script is popped and decoded.                        |
| `<sig> <pubKey>`                             | `OP_CHECKSIG`                                                          | Constant added.                                      |
| true                                         | Empty                                                                  | Signature is checked for top two stack items.        |

```

C'est peut-être mieux en vrai tableau markdown pour la mise en forme (et éventuellement traduction)

Author: Je voulais montrer un aperçu historique de comment la chose était présentée dans le message de Saberhagen sur le forum, mais peut-être que c'est mieux oui

Reviewer: Non c'est vrai tu as raison. Laissons le en image, c'est un détail 👍

>
> original: "My worry is bitcoin someday becomes corrupted. Developers: see this extra scrutiny as an opportunity to build a culture of openness. It is not at all bad."

Toutefois, ce n'est pas l'avis de Gavin Andresen qui choisit de trancher le débat en faveur du BIP 16. Il considère qu'il y a eu suffisamment de discussions, et ne veut pas que les développeurs continuent à se disputer sur les détails, à l'instar de « [la couleur de l'abri à vélos](https://en.wikipedia.org/wiki/Law_of_triviality) ». Le 30 janvier, il [déclare](https://buildingbitcoin.org/bitcoin-dev/log-2012-01-30.html#l-1766) sur IRC ne plus vouloir « perdre \[son\] temps avec le BIP 17 ».

Luke-Jr [crée](https://buildingbitcoin.org/bitcoin-dev/log-2012-01-30.html#l-2147) une page sur le wiki de Bitcoin recensant les positions des développeurs, à la suite de la suggestion d'un certain Userjj. Le refus de Gavin contre CHV (BIP 17) est formel, et contraste avec les autres techniciens qui approuvent Pay to Script ou qui considèrent que c'est mieux que rien.

> ![Avis des développeurs sur les différentes propositions, le 2 février 2012](assets/en/p2sh-votes-feb-2012.webp)

Reviewer: Pareil, on pourrait peut-être le passer en tableau. Quoi que, on perdrait les couleurs ici

Author: Pareil c'est le tableau qui était visible à l'époque sur le wiki. Celui-là je le garde par contre.

L'élément qui est à l'origine de la création de la première cryptomonnaie alternative est le problème des noms de domaine, un problème particulièrement important pour les personnes qui gravitent alors autour de Bitcoin. Le système des noms de domaine, c'est-à-dire des chaines de caractères facilement mémorisables comme apple.com ou bitcoin.org, est en effet centralisé entre les mains de l'ICANN (*Internet Corporation for Assigned Names and Numbers*), une société de droit californien à but non lucratif. De ce fait, ces noms de domaine sont [sujets](https://web.archive.org/web/20110617160904/http://btlj.org/2011/02/23/four-rounds-of-ice-domain-name-seizures-and-related-controversies-and-opposition/) à des saisies régulières, ce qui pose problème aux partisans de la liberté, qui [cherchent](https://web.archive.org/web/20101213122350/http://dot-p2p.org/index.php?title=Main_Page) des alternatives.

> Le triangle de Zooko, [théorisé](https://web.archive.org/web/20011020191610/http://zooko.com/distnames.html) en 2001 par le cypherpunk Bryce « Zooko » Wilcox, énonce qu'un identifiant ne peut réunir au maximum que deux des propriétés suivantes : la sécurité (un nom ne doit correspondre qu'à un seul et unique élément) ; la décentralisation (les autorités centralisées ne doivent pas être nécessaires pour déterminer à quel nom correspond quel élément) ; l'intelligibilité pour l'homme (le nom des éléments doit pouvoir avoir un sens et il doit être possible de le retenir). Mais à la suite l'émergence de Bitcoin, ce trilemme n'existe plus : il est possible de « réaliser la quadrature du triangle de Zooko » (original: "you can indeed use BitCoin to square Zooko's triangle") pour reprendre l'[expression](http://www.aaronsw.com/weblog/squarezooko) d'Aaron Schwartz.

Reviewer: ```suggestion
Le triangle de Zooko, [théorisé](https://web.archive.org/web/20011020191610/http://zooko.com/distnames.html) en 2001 par le cypherpunk Bryce « Zooko » Wilcox, énonce qu'un identifiant ne peut réunir au maximum que deux des propriétés suivantes : la sécurité (un nom ne doit correspondre qu'à un seul et unique élément) ; la décentralisation (les autorités centralisées ne doivent pas être nécessaires pour déterminer à quel nom correspond quel élément) ; l'intelligibilité pour l'homme (le nom des éléments doit pouvoir avoir un sens et il doit être possible de le retenir). Mais à la suite de l'émergence de Bitcoin, ce trilemme n'existe plus : il est possible de « réaliser la quadrature du triangle de Zooko » (original: "you can indeed use BitCoin to square Zooko's triangle") pour reprendre l'[expression](http://www.aaronsw.com/weblog/squarezooko) d'Aaron Swartz.
```

Reviewer: Aaron Swartz je pense

> « Le fait de rassembler tous les systèmes de quorum par preuve de travail dans une seule base de données ne passe pas à l'échelle. Bitcoin et BitDNS peuvent être utilisés séparément. \[...\] Les réseaux ont besoin d'avoir des destins différents. Les utilisateurs de BitDNS pourraient être extrêmement tolérants vis-à-vis de l'ajout de fonctionnalités permettant de traiter des données volumineuses, puisque peu de registraires de noms de domaine seraient nécessaires, tandis que les utilisateurs de Bitcoin pourraient devenir de plus en plus sectaires à propos de la limitation de la taille de la chaîne, pour que son accès reste facile pour beaucoup d'utilisateurs et pour les petits appareils. »
>
> original: "Piling every proof-of-work quorum system in the world into one dataset doesn't scale.  Bitcoin and BitDNS can be used separately.  \[...\] The networks need to have separate fates.  BitDNS users might be completely liberal about adding any large data features since relatively few domain registrars are needed, while Bitcoin users might get increasingly tyrannical about limiting the size of the chain so it's easy for lots of users and small devices."

> Un mois plus tard, au début de l'année 2011, une proposition similaire est [faite](http://www.aaronsw.com/weblog/squarezooko) par Aaron Schwartz, informaticien et hacktiviste américain, cofondateur de Reddit en 2005. Il a développé le concept indépendamment, inspiré par les remarques de son ami Dan Kaminsky. Son idée est [évoquée](https://bitcointalk.org/index.php?topic=1790.msg35755#msg35755) sur le forum de Bitcoin à ce moment-là. Il [tentera](https://web.archive.org/web/20110521214956/https://squaretriangle.jottit.com/faq) d'implémenter son concept dans un système appelé « Squaretriangle », et surnommé « Nakanames » en hommage au créateur de Bitcoin.

Reviewer: ```suggestion
Un mois plus tard, au début de l'année 2011, une proposition similaire est [faite](http://www.aaronsw.com/weblog/squarezooko) par Aaron Swartz, informaticien et hacktiviste américain, cofondateur de Reddit en 2005. Il a développé le concept indépendamment, inspiré par les remarques de son ami Dan Kaminsky. Son idée est [évoquée](https://bitcointalk.org/index.php?topic=1790.msg35755#msg35755) sur le forum de Bitcoin à ce moment-là. Il [tentera](https://web.archive.org/web/20110521214956/https://squaretriangle.jottit.com/faq) d'implémenter son concept dans un système appelé « Squaretriangle », et surnommé « Nakanames » en hommage au créateur de Bitcoin.
```

Entretemps, un effort pour mettre en œuvre BitDNS est fait. Le 4 décembre, Kiba [met en place](https://bitcointalk.org/index.php?topic=2072.msg26742#msg26742) un système de récompenses pour le premier qui implémentera la chose : une dizaine de membres du forum et le montant promis est de 3 500 BTC (seuls 442 BTC seront versés). Plusieurs propositions sont faites, dont le modèle [DomainChain](https://bitcointalk.org/index.php?topic=1790.msg27787#msg27787) de Ribuck et la [proposition](https://web.archive.org/web/20101219214135/https://privwiki.dreamhosters.com/wiki/Bitcoin_DNS_System_Proposal) de Theymos et Nanotube, qui tirent toutes deux avantage de la chaine de Bitcoin.

> Cet effort finit par mener à la création d'un système fonctionnel distinct de Bitcoin : Namecoin. La chaine est [lancée](https://bitcointalk.org/index.php?topic=6017.msg88356#msg88356) le 19 avril 2011 ([bloc 1](https://chainz.cryptoid.info/nmc/block.dws?1.htm)) par un certain Vincent Durham, qui utilise le pseudonyme Vinced sur le forum. Le logiciel est une version modifiée du logiciel de Bitcoin. Namecoin est ainsi très similaire à son grand frère : des unités appelées des namecoins sont émises à chaque bloc par des mineurs toutes les 10 minutes en moyenne, elles sont transférées par des transactions qui sont incluses dans les blocs, l'émission monétaire est réduite de moitié tous les 4 ans, etc. Les noms de domaines (qui [utiliseront](https://bitcointalk.org/index.php?topic=6851.msg126500#msg126500) l'extension `.bit`) sont des jetons spéciaux ne peuvent pas être mélangés entre eux (Vitalik Buterin les [nommera](https://ethereum.org/whitepaper/) « actifs non fongibles » (original: "non-fungible assets") en 2014) et qui sont déplacés avec les unités pour être liés à une adresse. La création et le renouvellement des noms sont réalisés grâce à un type spécial de transaction. Les domaines expirent au bout de 83 jours (12 000 blocs).

Reviewer: ```suggestion
Cet effort finit par mener à la création d'un système fonctionnel distinct de Bitcoin : Namecoin. La chaine est [lancée](https://bitcointalk.org/index.php?topic=6017.msg88356#msg88356) le 19 avril 2011 ([bloc 1](https://chainz.cryptoid.info/nmc/block.dws?1.htm)) par un certain Vincent Durham, qui utilise le pseudonyme Vinced sur le forum. Le logiciel est une version modifiée du logiciel de Bitcoin. Namecoin est ainsi très similaire à son grand frère : des unités appelées des namecoins sont émises à chaque bloc par des mineurs toutes les 10 minutes en moyenne, elles sont transférées par des transactions qui sont incluses dans les blocs, l'émission monétaire est réduite de moitié tous les 4 ans, etc. Les noms de domaine (qui [utiliseront](https://bitcointalk.org/index.php?topic=6851.msg126500#msg126500) l'extension `.bit`) sont des jetons spéciaux qui ne peuvent pas être mélangés entre eux (Vitalik Buterin les [nommera](https://ethereum.org/whitepaper/) « actifs non fongibles » (original: "non-fungible assets") en 2014) et qui sont déplacés avec les unités pour être liés à une adresse. La création et le renouvellement des noms sont réalisés grâce à un type spécial de transaction. Les domaines expirent au bout de 83 jours (12 000 blocs).
```

Reviewer: suppression de "s" "noms de domaine" + ajout du "qui" après "jetons spéciaux"

### La première vague des imitateurs

> Mais Namecoin n'est pas le seul système séparé à être lancée en 2011. La grande bulle de 2011 engendre en effet une [première vague](https://web.archive.org/web/20130722051950/https://github.com/ppcoin/ppcoin/wiki/History-of-cryptocurrency) de création de cryptomonnaies alternatives. Même si les intentions de leur inventeur ne peuvent pas être clairement identifiées, certains de ces projets se révèlent purement opportunistes, désirant profiter de l'enthousiasme vis-à-vis de Bitcoin, notamment par le biais de ce qu'on appelle préminage.

Reviewer: ```suggestion
Mais Namecoin n'est pas le seul système séparé à être lancé en 2011. La grande bulle de 2011 engendre en effet une [première vague](https://web.archive.org/web/20130722051950/https://github.com/ppcoin/ppcoin/wiki/History-of-cryptocurrency) de création de cryptomonnaies alternatives. Même si les intentions de leur inventeur ne peuvent pas être clairement identifiées, certains de ces projets se révèlent purement opportunistes, désirant profiter de l'enthousiasme vis-à-vis de Bitcoin, notamment par le biais de ce qu'on appelle préminage.
```

> Certains sont hautement expérimentaux, à l'instar bitcoins du réseau de test, et peuvent être réinitialisés à tout moment. C'est le cas des « [BeerTokens](https://web.archive.org/web/20110703101435/http://exchange.beertokens.info/) » ou des « [Weeds](https://web.archive.org/web/20110505081347/http://exchange.surething.biz/) » créés à l'aide du logiciel MultiCoin, [développé](https://bitcointalk.org/index.php?topic=24209.msg300830#msg300830) par un certain Scott Carlson (Sacarlson).

Reviewer: ```suggestion
Certains sont hautement expérimentaux, à l'instar des bitcoins du réseau de test, et peuvent être réinitialisés à tout moment. C'est le cas des « [BeerTokens](https://web.archive.org/web/20110703101435/http://exchange.beertokens.info/) » ou des « [Weeds](https://web.archive.org/web/20110505081347/http://exchange.surething.biz/) » créés à l'aide du logiciel MultiCoin, [développé](https://bitcointalk.org/index.php?topic=24209.msg300830#msg300830) par un certain Scott Carlson (Sacarlson).
```

La situation [donne l'idée](https://bitcointalk.org/index.php?topic=36425.msg448463#msg448463) à certains de développer I0coin, un fork d'Ixcoin « avec 0 bloc prégénéré » (original: "with 0 pregenerated blocks"). La chaine finit par être [lancée](https://bitcointalk.org/index.php?topic=36425.msg460733#msg460733) par un certain kr105 le 16 août. Elle a un temps de bloc de 5 minutes au lieu de 10. Elle [recueille](https://bitcointalk.org/index.php?topic=36425.msg461312#msg461312) une puissance de calcul de 1 TH/s dès le premier jour, chacun désirant se procurer des unités avant qu'elles ne prennent en valeur.

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: ```suggestion
En septembre, une autre cryptomonnaie opportuniste est [lancée](https://bitcointalk.org/index.php?topic=42417.msg516279#msg516279) par un membre du forum se faisant appeler Lolcust : il s'agit de GeistGeld (signifiant « monnaie spirituelle » en allemand). Elle est présentée comme une monnaie expérimentale, en tant que la génération de blocs est très rapide (le temps de bloc est de 15 secondes), générant probablement de nombreux blocs orphelins. Son créateur s'est réservé 7,7 millions d'unités pour lui-même \[source : Charlie Lee\].
```

Reviewer: "elle est présentée come une monnaie expérimentale" -> "comme"

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: ```suggestion
Sa première décision en tant que DPR est d'interdire la vente d'armes à feu sur Silk Road, celles-ci mettant mal à l'aise certains vendeurs sur Silk Road, et de [créer](https://antilop.cc/sr/users/dpr/threads/20120226-0702-A_brand_new_annonymous_market.html) une plateforme à part consacrée à ce type de produit. La plateforme sœur, nommée « The Armory », ouvre le 26 février 2012, mais l'expérience est de courte durée car elle [ferme](https://bitcointalk.org/index.php?topic=66587.msg1079466#msg1079466) en août faute de volume suffisant.
```

Reviewer: faute de volume suffisant

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: ```suggestion
Jusqu'alors, toutes les cryptomonnaies alternatives utilisaient le fonction de hachage de Bitcoin, le double SHA-256, pour le minage, parfois exploitant le minage combiné. Cela profite aux mineurs établis de la chaine de BTC, car ils disposent déjà du matériel pour générer des unités en quantité. Les nouveaux arrivants n'ont pas l'occasion de pouvoir miner ces nouvelles cryptomonnaies avec leurs processeurs centraux, ce qui rejoint le problème déjà existant de la variance.
```

Reviewer: Jusqu'à là -> Jusqu'alors

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: Percival* pas Perceval

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: communautés

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: ```suggestion
Un mois plus tard, Ross décide de changer son nom sur le forum et dans ses communications officielles. Il se présentait jusque là sous le pseudonyme « Silk Road ». Son nouveau nom, dévoilé sur le forum le 5 février, est Dread Pirate Roberts (rapidement abrégé en DPR). Il s'agit d'une référence au film *The Princess Bride*, une comédie sortie en 1987 parodiant les contes médiévaux, dans laquelle un personnage de Roberts est présenté comme un pirate sanguinaire. La révélation du film [est](https://www.youtube.com/watch?v=aHZGqBVBCRw) que le nom « Dread Pirate Roberts » n'est seulement qu'un titre et que le pirate actuel a hérité du titre de l'homme qui l'avait capturé, qui lui-même avait hérité du titre de son prédécesseur, etc., le nom étant censé « inspirer la peur nécessaire » (original: "inspiring the necessary fear") à ses ennemis.
```

Reviewer: communications *

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: ```suggestion
Une autre activité qui se développe bien durant l'année 2012 est le change entre le bitcoin et l'argent liquide physique, communément appelé le *cash*. Cette activité permet d'acheter ou de vendre du bitcoin de manière totalement confidentielle, sans laisse de trace de virement bancaire. Il y a plein de raisons de faire ce genre de transaction, mais la principale est évidemment liée à l'activité sur Silk Road. Les acheteurs peuvent se procurer du bitcoin pour obtenir de la drogue ; les vendeurs peuvent écouler leurs bitcoins en liquide pour payer leurs fournisseurs et avoir de quoi vivre. Cela vaut pour n'importe quelle activité plus ou moins illicite.
```

illicite

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: ```suggestion
L'activité de BTCKing est repérée rapidement par BitInstant et son intermédiaire (TrustCash), car plusieurs erreurs sont commises par ses clients lorsqu'ils font des dépôts. En particulier, un dépôt excède la limite de 1 000 $ fixée par l'entreprise, ce qui vaut à son premier compte d'être cloturé. Ayant compris ce que faisait BTCKing et conscient qu'il lui « apporte beaucoup de clients » (original: "He brings us a lot of business."), Charlie Shrem entre personnellement en contact avec lui, pour lui conseiller de procéder aux dépôts avec une autre adresse courriel. Mais cet arrangement ne plait pas à Gareth Nelson, son cofondateur, qui voti d'un mauvais œil cette activité. Dans une discussion avec lui en janvier 2012, Charlie [justifie](https://www.justice.gov/sites/default/files/usao-sdny/legacy/2015/03/25/Faiella%2C%20Robert%20M.%20and%20Charlie%20Shrem%20Complaint.pdf#page=17) son choix de la manière suivante :
```

conscient*

Reviewer: voti -> voit

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: enthouasiasmé -> enthousiasmé

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: accusées*

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: rudimentaire*

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: sont publiées*

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: éditions précédentes*

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: Seals With Clubs : à harmoniser. Parfois tu mets une majuscule à "with", parfois non

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: consacré

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: furent énormes * ? L'utilisation du futur me semble bizarre ici

Reviewer: Quoi que, finalement c'est cohérent, désolé

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: fournisseurs*

> Un troisième altcoin est SolidCoin [lancé](https://bitcointalk.org/index.php?topic=38453.msg471183#msg471183) publiquement le 21 août 2011 par un individu utilisant les pseudonymes CoinHunter et RealSolid. Il s'est [intéressé](https://bitcointalk.org/index.php?topic=15672.msg381911#msg381911) à Bitcoin pour son « potentiel de \[lui\] faire gagner de l'argent grâce à la spéculation » (original: "the potential to make \[him\] money through speculation"), pour sa facilité à être écoulé et pour son anonymat. SolidCoin se différencie de Bitcoin par un temps de bloc inférieur (3 minutes au lieu de 10), des frais fixes par transaction, et un algorithme d'ajustement modifié facilitant la diminution de la difficulté de minage par 

Reviewer: silte -> sites
